
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Genomic and Functional Characterization of Enterococcus faecalis Isolates Recovered From the International Space Station and Their Potential for Pathogenicity - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="DB2B70008E19D6A31470000002213801.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="frontmicrobio">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC7829349/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Frontiers in Microbiology">
<meta name="citation_title" content="Genomic and Functional Characterization of Enterococcus faecalis Isolates Recovered From the International Space Station and Their Potential for Pathogenicity">
<meta name="citation_author" content="Noelle C Bryan">
<meta name="citation_author_institution" content="Department of Earth, Atmospheric and Planetary Sciences, Massachusetts Institute of Technology, Cambridge, MA, United States">
<meta name="citation_author" content="Francois Lebreton">
<meta name="citation_author_institution" content="Department of Ophthalmology, Massachusetts Eye and Ear Infirmary, Boston, MA, United States">
<meta name="citation_author_institution" content="Walter Reed Army Institute of Research, Silver Spring, MD, United States">
<meta name="citation_author" content="Michael Gilmore">
<meta name="citation_author_institution" content="Department of Ophthalmology, Massachusetts Eye and Ear Infirmary, Boston, MA, United States">
<meta name="citation_author" content="Gary Ruvkun">
<meta name="citation_author_institution" content="Department of Molecular Biology, Massachusetts General Hospital, Boston, MA, United States">
<meta name="citation_author" content="Maria T Zuber">
<meta name="citation_author_institution" content="Department of Earth, Atmospheric and Planetary Sciences, Massachusetts Institute of Technology, Cambridge, MA, United States">
<meta name="citation_author" content="Christopher E Carr">
<meta name="citation_author_institution" content="Department of Earth, Atmospheric and Planetary Sciences, Massachusetts Institute of Technology, Cambridge, MA, United States">
<meta name="citation_author_institution" content="Department of Molecular Biology, Massachusetts General Hospital, Boston, MA, United States">
<meta name="citation_author_institution" content="Georgia Institute of Technology, Atlanta, GA, United States">
<meta name="citation_publication_date" content="2021 Jan 11">
<meta name="citation_volume" content="11">
<meta name="citation_firstpage" content="515319">
<meta name="citation_doi" content="10.3389/fmicb.2020.515319">
<meta name="citation_pmid" content="33505359">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC7829349/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC7829349/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC7829349/pdf/fmicb-11-515319.pdf">
<meta name="description" content="Enterococcus faecalis is a multidrug resistant, opportunistic human pathogen and a leading cause of hospital acquired infections. Recently, isolates have been recovered from the air and surfaces onboard the International Space Station (ISS). ...">
<meta name="og:title" content="Genomic and Functional Characterization of Enterococcus faecalis Isolates Recovered From the International Space Station and Their Potential for Pathogenicity">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Enterococcus faecalis is a multidrug resistant, opportunistic human pathogen and a leading cause of hospital acquired infections. Recently, isolates have been recovered from the air and surfaces onboard the International Space Station (ISS). ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC7829349/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://pmc.ncbi.nlm.nih.gov/search/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="7829349">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.3389/fmicb.2020.515319"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/fmicb-11-515319.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC7829349%2F%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/7829349/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/7829349/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC7829349/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-frontmicrobio.gif" alt="Frontiers in Microbiology logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Frontiers in Microbiology" title="Link to Frontiers in Microbiology" shape="default" href="http://www.frontiersin.org/microbiology/archive" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Front Microbiol</button></div>. 2021 Jan 11;11:515319. doi: <a href="https://doi.org/10.3389/fmicb.2020.515319" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.3389/fmicb.2020.515319</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://pmc.ncbi.nlm.nih.gov/search/?term=%22Front%20Microbiol%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Front%20Microbiol%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Front%20Microbiol%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Front%20Microbiol%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Genomic and Functional Characterization of <em>Enterococcus faecalis</em> Isolates Recovered From the International Space Station and Their Potential for Pathogenicity</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bryan%20NC%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Noelle C Bryan</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Noelle C Bryan</span></h3>
<div class="p">
<sup>1</sup>Department of Earth, Atmospheric and Planetary Sciences, Massachusetts Institute of Technology, Cambridge, MA, United States</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bryan%20NC%22%5BAuthor%5D" class="usa-link"><span class="name western">Noelle C Bryan</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lebreton%20F%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Francois Lebreton</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Francois Lebreton</span></h3>
<div class="p">
<sup>2</sup>Department of Ophthalmology, Massachusetts Eye and Ear Infirmary, Boston, MA, United States</div>
<div class="p">
<sup>3</sup>Walter Reed Army Institute of Research, Silver Spring, MD, United States</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lebreton%20F%22%5BAuthor%5D" class="usa-link"><span class="name western">Francois Lebreton</span></a>
</div>
</div>
<sup>2,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gilmore%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Michael Gilmore</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Michael Gilmore</span></h3>
<div class="p">
<sup>2</sup>Department of Ophthalmology, Massachusetts Eye and Ear Infirmary, Boston, MA, United States</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gilmore%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Michael Gilmore</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ruvkun%20G%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Gary Ruvkun</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Gary Ruvkun</span></h3>
<div class="p">
<sup>4</sup>Department of Molecular Biology, Massachusetts General Hospital, Boston, MA, United States</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ruvkun%20G%22%5BAuthor%5D" class="usa-link"><span class="name western">Gary Ruvkun</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zuber%20MT%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Maria T Zuber</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Maria T Zuber</span></h3>
<div class="p">
<sup>1</sup>Department of Earth, Atmospheric and Planetary Sciences, Massachusetts Institute of Technology, Cambridge, MA, United States</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zuber%20MT%22%5BAuthor%5D" class="usa-link"><span class="name western">Maria T Zuber</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Carr%20CE%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Christopher E Carr</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Christopher E Carr</span></h3>
<div class="p">
<sup>1</sup>Department of Earth, Atmospheric and Planetary Sciences, Massachusetts Institute of Technology, Cambridge, MA, United States</div>
<div class="p">
<sup>4</sup>Department of Molecular Biology, Massachusetts General Hospital, Boston, MA, United States</div>
<div class="p">
<sup>5</sup>Georgia Institute of Technology, Atlanta, GA, United States</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Carr%20CE%22%5BAuthor%5D" class="usa-link"><span class="name western">Christopher E Carr</span></a>
</div>
</div>
<sup>1,</sup><sup>4,</sup><sup>5,</sup><sup>*</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="aff1">
<sup>1</sup>Department of Earth, Atmospheric and Planetary Sciences, Massachusetts Institute of Technology, Cambridge, MA, United States</div>
<div id="aff2">
<sup>2</sup>Department of Ophthalmology, Massachusetts Eye and Ear Infirmary, Boston, MA, United States</div>
<div id="aff3">
<sup>3</sup>Walter Reed Army Institute of Research, Silver Spring, MD, United States</div>
<div id="aff4">
<sup>4</sup>Department of Molecular Biology, Massachusetts General Hospital, Boston, MA, United States</div>
<div id="aff5">
<sup>5</sup>Georgia Institute of Technology, Atlanta, GA, United States</div>
<div class="author-notes p">
<div class="fn" id="fn1"><p>Edited by: Rakesh Mogul, California State Polytechnic University, Pomona, United States</p></div>
<div class="fn" id="fn2"><p>Reviewed by: Kasthuri Venkateswaran, NASA Jet Propulsion Laboratory (JPL), United States; Madhan Tirumalai, University of Houston, United States</p></div>
<div class="fn" id="c001">
<sup>✉</sup><p class="display-inline">*Correspondence: Christopher E. Carr, <span>chrisc@mit.edu</span></p>
</div>
<div class="fn" id="fn004"><p>This article was submitted to Extreme Microbiology, a section of the journal Frontiers in Microbiology</p></div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2019 Nov 27; Accepted 2020 Dec 9; Collection date 2020.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>Copyright © 2021 Bryan, Lebreton, Gilmore, Ruvkun, Zuber and Carr.</div>
<p>This is an open-access article distributed under the terms of the Creative Commons Attribution License (CC BY). The use, distribution or reproduction in other forums is permitted, provided the original author(s) and the copyright owner(s) are credited and that the original publication in this journal is cited, in accordance with accepted academic practice. No use, distribution or reproduction is permitted which does not comply with these terms.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC7829349  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/33505359/" class="usa-link">33505359</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p><em>Enterococcus faecalis</em> is a multidrug resistant, opportunistic human pathogen and a leading cause of hospital acquired infections. Recently, isolates have been recovered from the air and surfaces onboard the International Space Station (ISS). Pangenomic and functional analyses were carried out to assess their potential impact on astronaut health. Genomes of each ISS isolate, and both clinical and commensal reference strains, were evaluated for their core and unique gene content, acquired antibiotic resistance genes, phage, plasmid content, and virulence traits. In order to determine their potential survival when outside of the human host, isolates were also challenged with three weeks of desiccation at 30% relative humidity. Finally, pathogenicity of the ISS strains was evaluated in the model organism <em>Caenorhabditis elegans.</em> At the culmination of this study, there were no defining signatures that separated known pathogenic strains from the more commensal phenotypes using the currently available resources. As a result, the current reliance on database information alone must be shifted to experimentally evaluated genotypic and phenotypic characteristics of clinically relevant microorganisms.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Keywords:</strong> International Space Station (ISS), <em>Enterococcus faecalis</em>, pangenome, antibiotic resistance, desiccation tolerance, pathogenicity</p></section></section><section id="S1"><h2 class="pmc_sec_title">Introduction</h2>
<p><em>Enterococcus faecalis</em> represent a core, yet quantitatively minor portion of the human gut microbiome (<a href="#B52" class="usa-link" aria-describedby="B52">Schloissnig et al., 2013</a>) that are well suited to persist outside of the host environment. Evidence recently emerged that the enterococci may have split from their last common ancestor at approximately the time animal life began to colonize terrestrial habitats (∼425 million years ago), and traits that promote survival and transmission in the exposed land environment selected for the characteristic ruggedness of the genus (<a href="#B37" class="usa-link" aria-describedby="B37">Lebreton et al., 2017</a>). Supporting that proposition, enterococci display significantly higher levels of resistance to a variety of antiseptics, salts, organic compounds, desiccation, and starvation than ancestral outgroups (<a href="#B37" class="usa-link" aria-describedby="B37">Lebreton et al., 2017</a>).</p>
<p>Because traits contributing to environmental persistence also contribute to persistence in the hospital environment, enterococci rank among leading causes of healthcare associated infections (<a href="#B68" class="usa-link" aria-describedby="B68">Weiner et al., 2016</a>). While certain strains of <em>E. faecalis</em> are pathogenic for hospitalized patients, <em>e.g</em>., MMH594 (<a href="#B27" class="usa-link" aria-describedby="B27">Huycke et al., 1991</a>) and V583 (<a href="#B49" class="usa-link" aria-describedby="B49">Sahm et al., 1989</a>), and possess genomes swollen to 3.3 Mb by the accretion of mobile genetic elements (MGEs), the genomes of commensal isolates are 25% smaller, as typified by the strain OG1RF (<a href="#B7" class="usa-link" aria-describedby="B7">Bourgogne et al., 2008</a>). In the antibiotic era, loss of CRISPR (clustered regularly interspaced short palindromic repeats) protection of the chromosome further facilitated the accumulation of additional antibiotic resistance by enterococci (<a href="#B21" class="usa-link" aria-describedby="B21">Fiore et al., 2019</a>). Several other MGEs have been directly linked to virulence factors (<a href="#B54" class="usa-link" aria-describedby="B54">Shankar et al., 2002</a>), however, the selective value of most mobile elements in clinical isolates of enterococci remains to be determined.</p>
<p>The ISS provides a unique opportunity to study the establishment of a microbiome in what originated as a clean, hermetically sealed built environment, where the influx of new microorganisms only occurs periodically with the arrival of new crew and supplies. The microbiome of the ISS has been extensively characterized in recent years by metagenomic (<a href="#B4" class="usa-link" aria-describedby="B4">Be et al., 2017</a>; <a href="#B35" class="usa-link" aria-describedby="B35">Lang et al., 2017</a>; <a href="#B57" class="usa-link" aria-describedby="B57">Singh et al., 2018b</a>; <a href="#B62" class="usa-link" aria-describedby="B62">Urbaniak et al., 2018</a>; <a href="#B6" class="usa-link" aria-describedby="B6">Blaustein et al., 2019</a>; <a href="#B43" class="usa-link" aria-describedby="B43">Mora et al., 2019</a>; <a href="#B55" class="usa-link" aria-describedby="B55">Sielaff et al., 2019</a>) and culture-based investigations (<a href="#B51" class="usa-link" aria-describedby="B51">Schiwon et al., 2013</a>; <a href="#B66" class="usa-link" aria-describedby="B66">Venkateswaran, 2017</a>; <a href="#B56" class="usa-link" aria-describedby="B56">Singh et al., 2018a</a>; <a href="#B6" class="usa-link" aria-describedby="B6">Blaustein et al., 2019</a>; <a href="#B43" class="usa-link" aria-describedby="B43">Mora et al., 2019</a>; <a href="#B58" class="usa-link" aria-describedby="B58">Sobisch et al., 2019</a>; <a href="#B63" class="usa-link" aria-describedby="B63">Urbaniak et al., 2019</a>), yet only recently have these effort been combined (<a href="#B43" class="usa-link" aria-describedby="B43">Mora et al., 2019</a>; <a href="#B55" class="usa-link" aria-describedby="B55">Sielaff et al., 2019</a>). The results are divided among those that report similarities to Earth built environments (<a href="#B51" class="usa-link" aria-describedby="B51">Schiwon et al., 2013</a>; <a href="#B43" class="usa-link" aria-describedby="B43">Mora et al., 2019</a>; <a href="#B55" class="usa-link" aria-describedby="B55">Sielaff et al., 2019</a>) including United States based spacecraft assembly cleanrooms (<a href="#B4" class="usa-link" aria-describedby="B4">Be et al., 2017</a>), and those that detected significant differences from terrestrial residences, the Human Microbiome Project (<a href="#B35" class="usa-link" aria-describedby="B35">Lang et al., 2017</a>), the Japan-based ISS analog module (<a href="#B28" class="usa-link" aria-describedby="B28">Ichijo et al., 2016</a>), and a French Guiana-based cleanroom (<a href="#B43" class="usa-link" aria-describedby="B43">Mora et al., 2019</a>). Despite the differences in methodologies and results, the detection of opportunistic human pathogens (<a href="#B51" class="usa-link" aria-describedby="B51">Schiwon et al., 2013</a>; <a href="#B4" class="usa-link" aria-describedby="B4">Be et al., 2017</a>; <a href="#B35" class="usa-link" aria-describedby="B35">Lang et al., 2017</a>; <a href="#B43" class="usa-link" aria-describedby="B43">Mora et al., 2019</a>; <a href="#B56" class="usa-link" aria-describedby="B56">Singh et al., 2018a</a>,<a href="#B57" class="usa-link" aria-describedby="B57">b</a>; <a href="#B55" class="usa-link" aria-describedby="B55">Sielaff et al., 2019</a>; <a href="#B58" class="usa-link" aria-describedby="B58">Sobisch et al., 2019</a>; <a href="#B63" class="usa-link" aria-describedby="B63">Urbaniak et al., 2019</a>) including <em>E. faecalis</em>, is a common phenomenon. With the exception of <a href="#B4" class="usa-link" aria-describedby="B4">Be et al. (2017)</a>, the previous metagenomic efforts were limited to characterizing diversity at the genus level or identifying antibiotic resistance or virulence gene content, and thereby lacked the genetic resolution to confidently identify pathogenic bacterial strains. While genome-based pathogenicity analyses have been performed for <em>Staphylococcus aureus</em> (<a href="#B6" class="usa-link" aria-describedby="B6">Blaustein et al., 2019</a>) and <em>Enterobacter bugandensis</em> (<a href="#B56" class="usa-link" aria-describedby="B56">Singh et al., 2018a</a>), only a single culture-based report has quantified pathogenicity (<em>Fusarium oxysporum</em> isolates from the ISS <a href="#B63" class="usa-link" aria-describedby="B63">Urbaniak et al., 2019</a>) in a host model. As a result, the ability to predict how opportunistic pathogens may impact crew health remains unclear.</p>
<p>In addition to characterizing the microbiome of the ISS, understanding the effects of spaceflight on bacterial physiology will be imperative for assessing the impacts on crew health. When <em>Pseudomonas aeruginosa</em> was exposed to spaceflight, there was an increase in the total cell viable numbers, biomass, biofilm thickness, and the cells produced a unique biofilm architecture not seen in ground controls of the species (<a href="#B40" class="usa-link" aria-describedby="B40">Mclean et al., 2001</a>; <a href="#B34" class="usa-link" aria-describedby="B34">Kim et al., 2013</a>). <a href="#B60" class="usa-link" aria-describedby="B60">Su et al. (2014)</a> documented growth curve, transcriptomic, and proteomic alterations (related to amino acid transport, metabolism, energy production, and conversion) in <em>Bacillus cereus</em> and <em>Serratia marcescens</em> during spaceflight (<a href="#B67" class="usa-link" aria-describedby="B67">Wang et al., 2014</a>). Carbon utilization profiles were altered among <em>S. marcescens</em> clones after flight, and transcriptomic and proteomic data revealed significant changes in metabolic functions (<a href="#B67" class="usa-link" aria-describedby="B67">Wang et al., 2014</a>). Additionally, <a href="#B70" class="usa-link" aria-describedby="B70">Wilson et al. (2007)</a> utilized proteomic analysis and expression profiles to demonstrate that changes in genomic regulation of <em>Salmonella typhymurium</em> were widely distributed, and virulence was increased in response to spaceflight. Although <a href="#B24" class="usa-link" aria-describedby="B24">Hammond et al. (2013)</a> report a significant decrease in <em>E. faecalis</em> OG1RF virulence when grown under spaceflight conditions, this was inferred by a reduction in the optical density of cultures grown in the presence of adult <em>Caenorhabditis elegans</em> rather than worm viability.</p>
<p>In addition to altered microbial physiological responses, astronaut health is also impacted by the spaceflight environment. Factors such as microgravity, physiological stress, isolation, abnormal circadian rhythms, and altered nutrition have been shown to weaken an astronaut’s immunity in as little as 10 to 15 days (<a href="#B17" class="usa-link" aria-describedby="B17">Crucian et al., 2013</a>), and these effects can persist up to 6 months post mission ISS (<a href="#B18" class="usa-link" aria-describedby="B18">Crucian et al., 2015</a>). This weakened immunity, combined with the prevalence of <em>E. faecalis</em> strains, or other potential pathogens including viruses (<a href="#B48" class="usa-link" aria-describedby="B48">Rooney et al., 2019</a>), could result in severe human health consequences on board the ISS, even in incidences of routine medicinal procedures (<em>i.e</em>., the introduction of a percutaneous catheter <a href="#B41" class="usa-link" aria-describedby="B41">Mermel, 2012</a>).</p>
<p>With the detection of enterococci, we sought to determine the risk they posed to residents within the ISS – specifically, do they represent pathogenic lineages commonly associated with infections in hospitals, or do they represent commensals shed into the environment as a consequence of human habitation? In order to determine if the challenging environment of the air and surfaces of the ISS have selected for pathogenic strains of <em>E. faecalis</em>, we coupled previously employed genomic data analyses (<a href="#B56" class="usa-link" aria-describedby="B56">Singh et al., 2018a</a>) with phenotypic characterization of <em>E. faecalis</em> isolates recovered onboard the ISS. By assessing their antibiotic resistance, desiccation tolerance, and pathogenicity in the well-established <em>C. elegans</em> model (<a href="#B23" class="usa-link" aria-describedby="B23">Garsin et al., 2001</a>), we sought to test existing methods for identification of strains of <em>E. faecalis</em> with a high likelihood of causing in-flight infections.</p></section><section id="S2"><h2 class="pmc_sec_title">Materials and Methods</h2>
<section id="S2.SS1"><h3 class="pmc_sec_title">Strains and Culture Conditions</h3>
<p>Bacterial isolates from the ISS (<a href="#F1" class="usa-link">Figure 1</a> and <a href="#T1" class="usa-link">Table 1</a>) were initially recovered on tryptic soy agar, and preliminary identification was performed using a VITEK identification system (bioMérieux, Hazelwood, MO, United States) as previously described (<a href="#B10" class="usa-link" aria-describedby="B10">Castro et al., 2004</a>). <em>E. faecalis</em> isolates OG1RF, MMH594, V583, and the genome sequences of <em>E. faecalis</em> 5952 and JH-1 were made available from the Gilmore lab strain collection. All additional available whole genomes of <em>E. faecalis</em> (N = 44) were obtained from the National Center for Biotechnology Information (NCBI).</p>
<figure class="fig xbox font-sm" id="F1"><h4 class="obj_head">FIGURE 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=7829349_fmicb-11-515319-g001.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/98bf/7829349/9dac790ab9d5/fmicb-11-515319-g001.jpg" loading="lazy" height="444" width="758" alt="FIGURE 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/F1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Isolate recovery locations on the International Space Station. Isolates (denoted 1-4) were recovered from the United States Lab and Node 3. Toilets (T) are located in Zvezda (Service Module) and Tranquility (Node 3).</p></figcaption></figure><section class="tw xbox font-sm" id="T1"><h4 class="obj_head">TABLE 1.</h4>
<div class="caption p"><p>Summary of genomic characterization of ISS and control strains of <em>Enterococcus faecalis</em>.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups" cellspacing="5" cellpadding="5">
<thead><tr>
<td valign="top" align="left" rowspan="1" colspan="1">Isolate (Accession Number)</td>
<td valign="top" align="left" rowspan="1" colspan="1">Sample Type</td>
<td valign="top" align="left" rowspan="1" colspan="1">Location</td>
<td valign="top" align="center" rowspan="1" colspan="1">Collection Date</td>
<td valign="top" align="center" rowspan="1" colspan="1">Expedition(Duration in days)</td>
<td valign="top" align="center" rowspan="1" colspan="1">Genome size (Mb)</td>
<td valign="top" align="center" rowspan="1" colspan="1">No. Unique Genes</td>
<td valign="top" align="center" rowspan="1" colspan="1">MLST</td>
<td valign="top" align="center" rowspan="1" colspan="1">Predicted Intact Phage</td>
<td valign="top" align="center" rowspan="1" colspan="1">Plasmids</td>
<td valign="top" align="center" rowspan="1" colspan="1">Pathogen Score(%)</td>
</tr></thead>
<tbody>
<tr>
<td valign="top" align="left" rowspan="1" colspan="1">ISS_1 (<a href="https://ncbi.nlm.nih.gov/nucleotide/CP046113" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">CP046113</a>)</td>
<td valign="top" align="left" rowspan="1" colspan="1">Air</td>
<td valign="top" align="left" rowspan="1" colspan="1">United States LAB</td>
<td valign="top" align="center" rowspan="1" colspan="1">7/15/2009</td>
<td valign="top" align="center" rowspan="1" colspan="1">20 (137)</td>
<td valign="top" align="center" rowspan="1" colspan="1">2.65*</td>
<td valign="top" align="center" rowspan="1" colspan="1">11</td>
<td valign="top" align="center" rowspan="1" colspan="1">875**</td>
<td valign="top" align="center" rowspan="1" colspan="1">No</td>
<td valign="top" align="center" rowspan="1" colspan="1">No</td>
<td valign="top" align="center" rowspan="1" colspan="1">84.4</td>
</tr>
<tr>
<td valign="top" align="left" rowspan="1" colspan="1">ISS_2 (<a href="https://ncbi.nlm.nih.gov/nucleotide/CP046112" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">CP046112</a>)</td>
<td valign="top" align="left" rowspan="1" colspan="1">Surface</td>
<td valign="top" align="left" rowspan="1" colspan="1">United States LAB</td>
<td valign="top" align="center" rowspan="1" colspan="1">4/15/11</td>
<td valign="top" align="center" rowspan="1" colspan="1">27 (70)</td>
<td valign="top" align="center" rowspan="1" colspan="1">2.93*</td>
<td valign="top" align="center" rowspan="1" colspan="1">39</td>
<td valign="top" align="center" rowspan="1" colspan="1">30</td>
<td valign="top" align="center" rowspan="1" colspan="1">3</td>
<td valign="top" align="center" rowspan="1" colspan="1">No</td>
<td valign="top" align="center" rowspan="1" colspan="1">85.4</td>
</tr>
<tr>
<td valign="top" align="left" rowspan="1" colspan="1">ISS_3 (<a href="https://ncbi.nlm.nih.gov/nucleotide/CP046111" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">CP046111</a>)</td>
<td valign="top" align="left" rowspan="1" colspan="1">Surface</td>
<td valign="top" align="left" rowspan="1" colspan="1">United States LAB</td>
<td valign="top" align="center" rowspan="1" colspan="1">4/15/11</td>
<td valign="top" align="center" rowspan="1" colspan="1">27 (70)</td>
<td valign="top" align="center" rowspan="1" colspan="1">2.94*</td>
<td valign="top" align="center" rowspan="1" colspan="1">39</td>
<td valign="top" align="center" rowspan="1" colspan="1">30</td>
<td valign="top" align="center" rowspan="1" colspan="1">3</td>
<td valign="top" align="center" rowspan="1" colspan="1">No</td>
<td valign="top" align="center" rowspan="1" colspan="1">85.4</td>
</tr>
<tr>
<td valign="top" align="left" rowspan="1" colspan="1">ISS_4 (<a href="https://ncbi.nlm.nih.gov/nucleotide/CP0461108-10" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">CP0461108-10</a>)</td>
<td valign="top" align="left" rowspan="1" colspan="1">Air</td>
<td valign="top" align="left" rowspan="1" colspan="1">Node 3</td>
<td valign="top" align="center" rowspan="1" colspan="1">10/23/13</td>
<td valign="top" align="center" rowspan="1" colspan="1">37 (61)</td>
<td valign="top" align="center" rowspan="1" colspan="1">2.91*</td>
<td valign="top" align="center" rowspan="1" colspan="1">25</td>
<td valign="top" align="center" rowspan="1" colspan="1">40</td>
<td valign="top" align="center" rowspan="1" colspan="1">1</td>
<td valign="top" align="center" rowspan="1" colspan="1">2</td>
<td valign="top" align="center" rowspan="1" colspan="1">85.4</td>
</tr>
<tr>
<td valign="top" align="left" rowspan="1" colspan="1">OG1RF<sup>‡</sup>
</td>
<td valign="top" align="left" rowspan="1" colspan="1">Oral</td>
<td valign="top" align="left" rowspan="1" colspan="1">NA</td>
<td valign="top" align="center" rowspan="1" colspan="1">≤ 1975</td>
<td valign="top" align="center" rowspan="1" colspan="1">NA</td>
<td valign="top" align="center" rowspan="1" colspan="1">2.74</td>
<td valign="top" align="center" rowspan="1" colspan="1">61</td>
<td valign="top" align="center" rowspan="1" colspan="1">1</td>
<td valign="top" align="center" rowspan="1" colspan="1">No</td>
<td valign="top" align="center" rowspan="1" colspan="1">No</td>
<td valign="top" align="center" rowspan="1" colspan="1">81.6</td>
</tr>
<tr>
<td valign="top" align="left" rowspan="1" colspan="1">MMH594<sup>‡</sup>
</td>
<td valign="top" align="left" rowspan="1" colspan="1">Blood</td>
<td valign="top" align="left" rowspan="1" colspan="1">NA</td>
<td valign="top" align="center" rowspan="1" colspan="1">1985</td>
<td valign="top" align="center" rowspan="1" colspan="1">NA</td>
<td valign="top" align="center" rowspan="1" colspan="1">3.25</td>
<td valign="top" align="center" rowspan="1" colspan="1">52</td>
<td valign="top" align="center" rowspan="1" colspan="1">6</td>
<td valign="top" align="center" rowspan="1" colspan="1">1</td>
<td valign="top" align="center" rowspan="1" colspan="1">3</td>
<td valign="top" align="center" rowspan="1" colspan="1">82.9</td>
</tr>
<tr>
<td valign="top" align="left" rowspan="1" colspan="1">V583<sup>‡</sup>
</td>
<td valign="top" align="left" rowspan="1" colspan="1">Blood</td>
<td valign="top" align="left" rowspan="1" colspan="1">NA</td>
<td valign="top" align="center" rowspan="1" colspan="1">1987</td>
<td valign="top" align="center" rowspan="1" colspan="1">NA</td>
<td valign="top" align="center" rowspan="1" colspan="1">3.36</td>
<td valign="top" align="center" rowspan="1" colspan="1">20</td>
<td valign="top" align="center" rowspan="1" colspan="1">6</td>
<td valign="top" align="center" rowspan="1" colspan="1">1</td>
<td valign="top" align="center" rowspan="1" colspan="1">3</td>
<td valign="top" align="center" rowspan="1" colspan="1">82.9</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/T1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="attrib1">
<p><em>* Results of hybrid assemblies (See <a href="#DS1" class="usa-link">Supplementary Information</a>).</em></p>
<p><em>** Single nucleotide variant of the nearest ST.</em></p>
<p><em><sup>‡</sup><a href="#B39" class="usa-link" aria-describedby="B39">McBride et al. (2007)</a>.</em></p>
<p><em>NA: Not applicable.</em></p>
</div></div></section><p>Except where described below, cultures of <em>E. faecalis</em> OG1RF, MMH594, V583, and the ISS isolates (<a href="#T1" class="usa-link">Table 1</a>) were grown aerobically with shaking (250 RPM) at 37°C in brain heart infusion (BHI; Difco<sup>TM</sup>, cat. no: 237500) broth. Growth kinetics for each isolate was assessed using a Synergy 2 microplate reader (Bio-Tek Instruments Inc.), measuring optical density at 620 nm. Minimum inhibitory concentration (MIC) for various antibiotics was evaluated in both Mueller-Hinton (MH) broth or in BHI, as further detailed below. For the desiccation experiments, isolates were grown overnight in a chemically defined medium (CDM; <a href="#B26" class="usa-link" aria-describedby="B26">Hussain et al., 1991</a>) and plated onto M9 (Difco<sup>TM</sup>, cat. no: 237500) agar plates amended with 1.0% glucose. To perform the <em>C. elegans</em> pathogenicity assays, cultures of <em>Escherichia coli</em> OP50 were prepared on nematode growth media (NGM) agar plates as previously described (<a href="#B47" class="usa-link" aria-describedby="B47">Powell and Ausubel, 2008</a>).</p></section><section id="S2.SS2"><h3 class="pmc_sec_title">Genome Sequencing of ISS Strains</h3>
<p>For preparation of DNA, isolates were grown overnight in BHI broth. Cells were lysed using lysozyme (50 μg mL<sup>–1</sup>) and mutanolysin (2500 U mL<sup>–1</sup>), and the total DNA was isolated using the DNeasy Blood and Tissue Kit (Qiagen©, cat. no: 69504). Purified DNA was used for preparing libraries, which were then sequenced on Illumina HiSeq (250 nucleotide paired end reads) as previously described (<a href="#TS1" class="usa-link">Supplementary Table S1</a> in <a href="#B37" class="usa-link" aria-describedby="B37">Lebreton et al., 2017</a>).</p>
<p>Additionally, a MinION (Oxford Nanopore Technologies©; ONT) library was prepared using the “one-pot” barcoding protocol, as developed by Josh Quick and the Loman Lab<sup><a href="#footnote1" class="usa-link">1</a></sup>. For each isolate, 24 μL of AMPure XP (Beckman Coulter<sup>®</sup> Life Sciences cat. no.: A63880) eluate was used as input for library preparation. This was incubated at room temperature (RT) for 5 min, 65°C for 5 min, then placed on ice for 30 s. Samples were barcoded using the Rapid Barcoding Kit (ONT cat. no.: SQK-RBK001) and each was incubated at RT for 10 min, 70°C for 5 min, then placed on ice. The reaction product was then pooled with other samples (<em>N</em> = 4 total) in a clean 1.5 mL low-bind Eppendorf tube, before addition of 26.75 μl AMPure XP beads per sample. This was incubated at RT for 5 min, placed on the magnet rack until clear, and the supernatant removed. The beads were washed with 200 μl 70% ethanol, incubated for 30 s, and the supernatant removed (2X) before spinning down and removal of the residual 70% ethanol. After air drying for 1 min, the beads were resuspended in 31 μl elution buffer (EB, 10 mM Tris–HCl pH 8), incubated off the magnet rack for 5 min, returned to the rack. DNA was quantified by fluorescence (ThermoFisher Scientific© Qubit 3.0 with Qubit dsDNA HS Assay Kit; cat. no.: <a href="https://www.ncbi.nlm.nih.gov/protein/Q32854" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Q32854</a>) and the sequencing adaptors were ligated (ONT cat. no.: SQK-LSK109) following the manufacturer’s protocols. This tube was incubated at RT for 10 min, 45.5 μl AMPure XP beads added, incubated for another 5 min, placed on magnet rack until clear, and the supernatant removed. Next, 150 μl ABB was added and the beads resuspended by flicking, before placing the tube back on the magnet rack until clear. The supernatant was removed and the ABB wash repeated before spinning down the tube and removing the residual supernatant. Next, 12 ul EB was added and the bead resuspended by flicking before incubation at RT for 5 min. The tube was placed again on the magnet rack until clear, and the elution library was loaded on to an ONT MinION flow cell (R9.4.1) according to the manufacturer’s protocols and sequenced using MinKNOW (v1.11.5) with live basecalling (<a href="#TS2" class="usa-link">Supplementary Table S2</a>). Output fastq files were concatenated and adaptors were removed and debarcoded with PoreChop (v0.2.3<sup><a href="#footnote2" class="usa-link">2</a></sup>). Reads were quality filtered (≥ Q9) using NanoFilt (v2.2.0), and NanoPlot (v1.13.0) was then used to characterize sequencing datasets (<a href="#B19" class="usa-link" aria-describedby="B19">De Coster et al., 2018</a>).</p></section><section id="S2.SS3"><h3 class="pmc_sec_title">Genome Assembly</h3>
<p>Unicycler (v0.4.4) was used for long-read and hybrid assembly using default parameters, with the initial long-read assembly specified via the –existing_long_read_assembly flag for hybrid assembly (<a href="#B69" class="usa-link" aria-describedby="B69">Wick et al., 2017</a>). BWA (v0.7.17-r1188 with -x ont2d flag; docker container alexcoppe/bwa:latest) was used to map long reads onto the hybrid assembly to inspect assemblies and verify lack of evidence of mis-assemblies based on coverage. CheckM (v1.0.7) was used to evaluate completeness of final hybrid assemblies described in <a href="#T1" class="usa-link">Table 1</a> (<a href="#B45" class="usa-link" aria-describedby="B45">Parks et al., 2015</a>).</p></section><section id="S2.SS4"><h3 class="pmc_sec_title">Pangenomic and Phylogenetic Analysis</h3>
<p>Genomes were annotated using Prokka (v.1.14.6; docker container quay.io/biocontainers/prokka:1.14.6–pl526_0 <a href="#B53" class="usa-link" aria-describedby="B53">Seemann, 2014</a>), and the pangenome analysis was performed using Roary (v3.12.0; docker container staphb/roary:3.12.0 <a href="#B44" class="usa-link" aria-describedby="B44">Page et al., 2015</a>). Genes that were present in a single isolate were defined as unique genes and, for strains utilized in this study, their identities were further examined by BLASTn (<a href="#B1" class="usa-link" aria-describedby="B1">Altschul et al., 1990</a>) for comparison to the NCBI non-redundant database. Based on DNA sequence comparison, an <em>E. faecalis</em> phylogenetic tree (<a href="#F2" class="usa-link">Figure 2A</a>) was constructed from the Roary-determined core gene alignment (1.69 Mb) using FastTree (docker container staphb/fasttree:2.1.11) using the General Time Invariant (GTR) model, and visualized using IcyTree (<a href="#B65" class="usa-link" aria-describedby="B65">Vaughan, 2017</a>). For a list of all genomes and gene presence/absence data see <a href="#DS1" class="usa-link">Supplementary Data File S1</a>.</p>
<figure class="fig xbox font-sm" id="F2"><h4 class="obj_head">FIGURE 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=7829349_fmicb-11-515319-g002.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/98bf/7829349/260760881b3f/fmicb-11-515319-g002.jpg" loading="lazy" height="411" width="758" alt="FIGURE 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/F2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Whole genome analysis of <em>Enterococcus faecalis</em>. <strong>(A)</strong> Phylogenetic analysis of <em>E. faecalis</em> strains was determined based on FastTree generalized-time-reversible (GTR) analysis of SNPs present in the 1,844 core genes as determined by Roary (<a href="#B44" class="usa-link" aria-describedby="B44">Page et al., 2015</a>) and visualized with IcyTree (<a href="#B65" class="usa-link" aria-describedby="B65">Vaughan, 2017</a>). The alignment, when corrected for gaps and ambiguous bases, was 1.69 Mb. Branch support was 100% except as indicated. Isolates used in this study are indicated in <strong>bold</strong>. Asterisk (*) denotes an unfinished genome assembly (contigs). Clinical isolates that are phylogenetic neighbors of the ISS isolates are tagged with the biohazard symbol. <strong>(B)</strong> Genome size and gene count, the latter as estimated by Prokka (<a href="#B53" class="usa-link" aria-describedby="B53">Seemann, 2014</a>). <strong>(C)</strong> Gene presence/absence as estimated by Roary for the total of 9300 genes identified across the 51 genomes. A histogram of these data is provided as <a href="#FS1" class="usa-link">Supplementary Figure S1</a>.</p></figcaption></figure></section><section id="S2.SS5"><h3 class="pmc_sec_title">Comparative Genomic Analysis</h3>
<p>Multilocus sequence typing (MLST) was performed by evaluating seven <em>E. faecalis</em> genes (<em>aroE</em>, <em>gdh</em>, <em>gki</em>, <em>gyd</em>, <em>pstS</em>, <em>xpt</em>, and <em>yqiL</em>) and sequence types (STs) assigned based on the alleles present in each genome (<a href="#B36" class="usa-link" aria-describedby="B36">Larsen et al., 2012</a>). The potential presence of plasmids was assessed using PlasmidFinder (<a href="#B9" class="usa-link" aria-describedby="B9">Carattoli et al., 2014</a>). VirFinder was used to detect potential <em>E. faecalis</em> virulence factor genes (<a href="#B33" class="usa-link" aria-describedby="B33">Joensen et al., 2014</a>), and PathogenFinder provided a predictive score for each strain’s potential pathogenicity for humans (<a href="#B15" class="usa-link" aria-describedby="B15">Cosentino et al., 2013</a>). Additionally, the web server PHAge Search Tool – Enhanced Release (PHASTER) was used to detect the presence of phage sequences within bacterial genomic data (<a href="#B2" class="usa-link" aria-describedby="B2">Arndt et al., 2016</a>), and the Comprehensive Antibiotic Resistance Database (CARD) was used to identify genes potentially associated with antimicrobial resistance (<a href="#B31" class="usa-link" aria-describedby="B31">Jia et al., 2016</a>). The presence of CRISPR (clustered regularly interspaced short palindromic repeats) arrays and Cas (CRISPR associated proteins) was detected using the online tool, CRISPRCasFinder (<a href="#B16" class="usa-link" aria-describedby="B16">Couvin et al., 2018</a>). Capsule type polymorphisms were determined as described by <a href="#B39" class="usa-link" aria-describedby="B39">McBride et al. (2007)</a>. Briefly the presence, or absence, of genes in the locus (EF2485 to EF2495) were verified in the genomes of each isolate. CPS type 1 strains are defined as isolates with <em>cpsA</em> and <em>B</em>, and <em>hcp1</em> (EF2484), while CPS type 2 strains consist of the full <em>cps</em> locus and <em>hcp1</em> (EF2484 to EF2495). CPS type 5 strains are similar to the type 2 gene profile, but do not contain <em>cpsF</em> (EF0090).</p></section><section id="S2.SS6"><h3 class="pmc_sec_title">Minimum Inhibitory Concentration of Antibiotics</h3>
<p>Antibiotic MIC was determined by broth microdilution as described (<a href="#B13" class="usa-link" aria-describedby="B13">CLSI, 2017</a>). Briefly, overnight cultures of isolates were diluted and 10<sup>4</sup> cells in MH broth were inoculated into each well of 96 well plates containing serial 2-fold dilutions of antibiotics. Maximum concentration tested: penicillin (10 μg mL<sup>–1</sup>), amoxicillin (10 μg mL<sup>–1</sup>), ampicillin (10 μg mL<sup>–1</sup>), oxacillin (256 μg mL<sup>–1</sup>), erythromycin (50 μg mL<sup>–1</sup>), and tetracycline (64 μg mL<sup>–1</sup>). This procedure was repeated in BHI (<a href="#B39" class="usa-link" aria-describedby="B39">McBride et al., 2007</a>), and isolates were additionally evaluated for high level aminoglycoside resistance (HLGR) with: gentamicin, streptomycin, and kanamycin (each at a maximum concentration of 2 mg mL<sup>–1</sup>). Plates were incubated at 37°C and growth was monitored at 24 and 48 h. Wells containing the lowest antibiotic concentration lacking growth after 48 h were designated the MIC for the respective antibiotics.</p></section><section id="S2.SS7"><h3 class="pmc_sec_title">Desiccation Survival</h3>
<p>Survival to desiccation was assessed as previously described (<a href="#B37" class="usa-link" aria-describedby="B37">Lebreton et al., 2017</a>; <a href="#B100" class="usa-link" aria-describedby="B100">Bryan et al., 2019</a>) with the following modifications. <em>E. faecalis</em> isolates were harvested from stationary phase by centrifugation, and washed twice with sterile (autoclaved, 0.22 μm filtered) water (sH<sub>2</sub>O). Prior to desiccation, the concentrated cells were resuspended in CDM, and a 10 μL sample was track diluted (<a href="#B30" class="usa-link" aria-describedby="B30">Jett et al., 1997</a>) onto M9 medium (M9; Amresco<sup>®</sup>, cat. no.: J863-500G) amended with 0.1% glucose agar, and incubated at 37°C. After ∼24 h, the number of CFUs recovered from 10 μL track dilutions prepared according to <a href="#B30" class="usa-link" aria-describedby="B30">Jett et al. (1997)</a> were used to determine the original concentration of cells mL<sup>–1</sup> (<em>t</em> = −1). Aliquots of cell suspensions (50 μL) were spotted onto autoclaved coverslips, and placed into the desiccation chamber containing Drierite<sup>®</sup> (≥ 98% CaSO<sub>4</sub>, W. A. Hammond Drierite Company, Ltd., cat. no.: 778-18-9) until the liquid evaporated (<em>t</em> = <em>0</em>, 24 h). The relative humidity (RH) in the chamber was monitored using a digital hygrometer, and maintained between 30-40% by adding additional Drierite<sup>®</sup>. Three replicates were prepared for each isolate at each timepoint. The dried cells were removed from the coverslips by rinsing and resuspending the material with a pipet into 1 mL of sH<sub>2</sub>O. To ensure the samples were homogenously resuspended, rehydrated cells were incubated at 37°C for 1 h while shaking (225 RPM), and vortexed for 15 minutes. The cells were then serially diluted into sH<sub>2</sub>O and 10 μL track dilutions were plated onto M9 agar. At the <em>t</em> = 0 time point, the number of CFUs recovered from the substrate was designated as the starting population size that survived the initial drying on the coverslip (<em>N</em><sub>0</sub>). Subsequent samples were removed from the desiccation chamber every three days and rehydrated as described above to determine the number of surviving CFUs over time (<em>N</em>). The surviving fraction of cells was determined from the ratio of <em>N/N<sub>0</sub></em>. Each time point was evaluated when <em>N</em> was ≥ 30 CFU.</p></section><section id="S2.SS8"><h3 class="pmc_sec_title">Pathogenicity of <em>E. faecalis</em> isolates in a <em>C. elegans</em> model</h3>
<p>Nematode killing assays were performed as previously described (<a href="#B71" class="usa-link" aria-describedby="B71">Yuen and Ausubel, 2018</a>). Briefly, <em>C. elegans fer-15;fem-1</em> (CF512) worms were grown on NGM inoculated with <em>E. coli</em> strain OP50 and incubated at 15°C until they reached the L4 stage. Approximately 30–40 of the L4 stage worms were then transferred to BHI agar plates (amended with kanamycin, 10 μg mL<sup>–1</sup>) inoculated with 100 μL aliquots of each log phase <em>E. faecalis</em> isolate. The plates were incubated at 25°C and worms were examined for viability. Worms that did not exhibit movement nor respond to physical touch were scored as dead and removed from the plate. Moving worms were scored as alive and also counted.</p></section><section id="S2.SS9"><h3 class="pmc_sec_title">Statistical Analyses</h3>
<p>Statistical analyses for the desiccation experiments were performed using JMP<sup>®</sup> Pro v.14.3 (<a href="#B32" class="usa-link" aria-describedby="B32">JMP<sup>®</sup>, 2019</a>). Prior to evaluating the data for significant differences, the data were assessed for equal variances. Subsequent one-way ANOVAs were performed to determine the significant differences of desiccation survival at different timepoints and among individual isolates. <em>Post hoc</em> comparisons for time were performed using parametric comparisons with a control using Dunnett’s method (<em>N</em> = 18 for each isolate, with the exception of <em>t</em> = 12 d where <em>N</em> = 17). The mean <em>N/N<sub>0</sub></em> for each isolate at <em>t</em> = 21 d was evaluated using the non-parametric comparison with a control performed following the Steel method (<em>N</em> = 3 for each isolate). Individual comparisons for survival at two timepoints were performed using a homoscedastic Student’s <em>t-</em> test.</p>
<p>The survival of <em>C. elegans</em> on each <em>E. faecalis</em> isolate was performed in triplicate. A Kaplan-Meier log rank analysis was performed to evaluate the survival curves using OASIS-2 (<a href="#B25" class="usa-link" aria-describedby="B25">Han et al., 2016</a>) as previously described Yuen (<a href="#B71" class="usa-link" aria-describedby="B71">Yuen and Ausubel, 2018</a>). The data were determined to follow the normal distribution using the Shapiro-Wilk test (<a href="#B25" class="usa-link" aria-describedby="B25">Han et al., 2016</a>), and pairwise comparisons with Bonferroni-corrected <em>p</em>-values &lt; 0.05 were considered significant. Data are reported as the average and the standard error (± SE) of the number of worms scored, and is reported in <a href="#T2" class="usa-link">Table 2</a>.</p>
<section class="tw xbox font-sm" id="T2"><h4 class="obj_head">TABLE 2.</h4>
<div class="caption p"><p>Analysis of the survival of <em>Caenorhabditis elegans</em> infected with the ISS and control strains of <em>Enterococcus faecalis</em>.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups" cellspacing="5" cellpadding="5">
<thead>
<tr>
<td valign="top" align="left" rowspan="1" colspan="1">Isolate</td>
<td valign="top" align="center" rowspan="1" colspan="1"><em>N*</em></td>
<td valign="top" align="center" colspan="3" rowspan="1">Restricted Mean</td>
<td valign="top" align="center" rowspan="1" colspan="1">LT<sub>50</sub>
</td>
<td valign="top" align="center" rowspan="1" colspan="1">Pairwise Comparisons**</td>
</tr>
<tr>
<td valign="top" align="center" rowspan="1" colspan="1"></td>
<td valign="top" align="center" rowspan="1" colspan="1"></td>
<td valign="top" align="center" colspan="3" rowspan="1"><hr></td>
</tr>
<tr>
<td rowspan="1" colspan="1"></td>
<td rowspan="1" colspan="1"></td>
<td valign="top" align="center" rowspan="1" colspan="1">Days</td>
<td valign="top" align="center" rowspan="1" colspan="1">SE</td>
<td valign="top" align="center" rowspan="1" colspan="1">95% C.I.</td>
<td rowspan="1" colspan="1"></td>
<td rowspan="1" colspan="1"></td>
</tr>
</thead>
<tbody>
<tr>
<td valign="top" align="left" rowspan="1" colspan="1">ISS_1</td>
<td valign="top" align="center" rowspan="1" colspan="1">37.0</td>
<td valign="top" align="center" rowspan="1" colspan="1">9.00</td>
<td valign="top" align="center" rowspan="1" colspan="1">0.66</td>
<td valign="top" align="center" rowspan="1" colspan="1">7.44–10.0</td>
<td valign="top" align="center" rowspan="1" colspan="1">8.13</td>
<td valign="top" align="center" rowspan="1" colspan="1">BC</td>
</tr>
<tr>
<td valign="top" align="left" rowspan="1" colspan="1">ISS_2</td>
<td valign="top" align="center" rowspan="1" colspan="1">33.0</td>
<td valign="top" align="center" rowspan="1" colspan="1">13.0</td>
<td valign="top" align="center" rowspan="1" colspan="1">0.53</td>
<td valign="top" align="center" rowspan="1" colspan="1">12.5–14.5</td>
<td valign="top" align="center" rowspan="1" colspan="1">12.9</td>
<td valign="top" align="center" rowspan="1" colspan="1">AB</td>
</tr>
<tr>
<td valign="top" align="left" rowspan="1" colspan="1">ISS_3</td>
<td valign="top" align="center" rowspan="1" colspan="1">29.0</td>
<td valign="top" align="center" rowspan="1" colspan="1">13.0</td>
<td valign="top" align="center" rowspan="1" colspan="1">0.59</td>
<td valign="top" align="center" rowspan="1" colspan="1">12.0–14.3</td>
<td valign="top" align="center" rowspan="1" colspan="1">13.1</td>
<td valign="top" align="center" rowspan="1" colspan="1">ABC</td>
</tr>
<tr>
<td valign="top" align="left" rowspan="1" colspan="1">ISS_4</td>
<td valign="top" align="center" rowspan="1" colspan="1">35.0</td>
<td valign="top" align="center" rowspan="1" colspan="1">10.0</td>
<td valign="top" align="center" rowspan="1" colspan="1">0.70</td>
<td valign="top" align="center" rowspan="1" colspan="1">9.06–11.8</td>
<td valign="top" align="center" rowspan="1" colspan="1">10.4</td>
<td valign="top" align="center" rowspan="1" colspan="1">BC</td>
</tr>
<tr>
<td valign="top" align="left" rowspan="1" colspan="1">OG1RF</td>
<td valign="top" align="center" rowspan="1" colspan="1">42.0</td>
<td valign="top" align="center" rowspan="1" colspan="1">10.0</td>
<td valign="top" align="center" rowspan="1" colspan="1">0.48</td>
<td valign="top" align="center" rowspan="1" colspan="1">9.5–11.3</td>
<td valign="top" align="center" rowspan="1" colspan="1">10.0</td>
<td valign="top" align="center" rowspan="1" colspan="1">BC</td>
</tr>
<tr>
<td valign="top" align="left" rowspan="1" colspan="1">MMH594</td>
<td valign="top" align="center" rowspan="1" colspan="1">35.0</td>
<td valign="top" align="center" rowspan="1" colspan="1">9.00</td>
<td valign="top" align="center" rowspan="1" colspan="1">0.50</td>
<td valign="top" align="center" rowspan="1" colspan="1">8.4–10.3</td>
<td valign="top" align="center" rowspan="1" colspan="1">8.88</td>
<td valign="top" align="center" rowspan="1" colspan="1">BC</td>
</tr>
<tr>
<td valign="top" align="left" rowspan="1" colspan="1">V583</td>
<td valign="top" align="center" rowspan="1" colspan="1">34.0</td>
<td valign="top" align="center" rowspan="1" colspan="1">11.0</td>
<td valign="top" align="center" rowspan="1" colspan="1">0.91</td>
<td valign="top" align="center" rowspan="1" colspan="1">8.71–12.3</td>
<td valign="top" align="center" rowspan="1" colspan="1">9.00</td>
<td valign="top" align="center" rowspan="1" colspan="1">ABC</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/T2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="attrib5"><p><em>The data are reported as the restricted mean (± SE) of three replicates, with the exception of ISS_1 (<em>N</em> = 2 at t = 9 <em>d</em>). * Starting number of worms. ** Pairwise analysis of each Kaplan-Meier survival curve using the log-rank test (<a href="#B25" class="usa-link" aria-describedby="B25">Han et al., 2016</a>). Those that were not significantly different from each other are indicated with a capital letter, with A being the least pathogenic (<em>p</em> &gt; 0.05).</em></p></div></div></section></section></section><section id="S3"><h2 class="pmc_sec_title">Results</h2>
<section id="S3.SS1"><h3 class="pmc_sec_title">Isolates Recovered From the ISS</h3>
<p>The <em>E. faecalis</em> isolates recovered from air samples (ISS_1 and ISS_4) were both collected mid-module. ISS_1 was isolated from air samples collected in the United States Lab in 2009, and ISS_4 was recovered from the air in Node 3 in 2013 (<a href="#F1" class="usa-link">Figure 1</a>). ISS_2 and ISS_3, were two isolates recovered in 2011 from the same location, United States Lab handrail surfaces, approximately 14 feet apart. One was collected from the aft end of the module (at the interface with Node 1), and the other was recovered mid-module. The United States Lab was launched in 2001, and houses the various science payloads conducting research onboard the ISS (<a href="#B57" class="usa-link" aria-describedby="B57">Singh et al., 2018b</a>). Node 3, launched in 2010, and houses the air revitalization and water recovery systems, as well as amenities for crew hygiene, the restrooms, and exercise devices (<a href="#B57" class="usa-link" aria-describedby="B57">Singh et al., 2018b</a>). With preliminary identification based solely on the VITEK identification system, the first task for the isolate characterization was to create a fully sequenced genome for each isolate (<a href="#T1" class="usa-link">Table 1</a>).</p></section><section id="S3.SS2"><h3 class="pmc_sec_title">Pangenome Analysis</h3>
<p>With the complete genomes for each isolate assembled, several web-based tools were utilized for genomic characterization. First, seven gene loci were evaluated for their respective MLST using the web-based MLST server (<a href="#T1" class="usa-link">Table 1</a>; <a href="#B36" class="usa-link" aria-describedby="B36">Larsen et al., 2012</a>). ISS_1 was determined to be a single locus variant of sequence type (ST) 875, with a single nucleotide polymorphism (SNP) in the 583 bases of <em>pstS</em>. Isolates ISS_2 and ISS_3 were identified as ST30. ISS_4 was classified as ST40, a commonly recovered ST and a member of one of the largest clonal clusters, CC40 (<a href="#B39" class="usa-link" aria-describedby="B39">McBride et al., 2007</a>). A clonal cluster is defined as a ST consisting of three or more isolates (<a href="#B39" class="usa-link" aria-describedby="B39">McBride et al., 2007</a>). CC40 is comprised of ST40 and ST114, many of which were isolated from clinical sources (<a href="#B39" class="usa-link" aria-describedby="B39">McBride et al., 2007</a>).</p>
<p>Based on the ST data described above, representative type strains were chosen for each as a comparator for each ISS ST (<a href="#B39" class="usa-link" aria-describedby="B39">McBride et al., 2007</a>), and a core genome of 1,844 genes shared by all 51 strains was defined (<a href="#B44" class="usa-link" aria-describedby="B44">Page et al., 2015</a>). An alignment of the core genome was then used to construct the phylogenetic tree (<a href="#F2" class="usa-link">Figure 2A</a>). Phylogenetic proximity of the selected type strains to the ISS isolates validated the selection based on ST. Genome size and gene count (<a href="#F2" class="usa-link">Figure 2B</a>) show patterns of genomic expansion and loss within clades; for example, within the clade containing V583 a general pattern of larger genome size and higher gene count is evident, yet nearby ISS_1 has a more streamlined genome in comparison to its neighbors. Genes are classified into core, soft core, shell, and cloud on the basis of the number of isolates sharing each gene (<a href="#F2" class="usa-link">Figure 2C</a>; <a href="#FS1" class="usa-link">Supplementary Figure S1</a>) and the Roary-estimated patterns of gene presence and absence highlight potential patterns of gene loss or gain (<a href="#DS1" class="usa-link">Supplementary Data File S1</a>).</p>
<p>Unique chromosomal genes, defined as those genes that were present in a single isolate, were non-uniformly distributed across each isolate genome (<a href="#DS1" class="usa-link">Supplementary Data File S1</a>). ISS_1 (Accession no.: <a href="https://ncbi.nlm.nih.gov/nucleotide/CP046113" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">CP046113</a>) contained the fewest number of unique genes (11 genes), 9 of which were designated hypothetical by Prokka (<a href="#B53" class="usa-link" aria-describedby="B53">Seemann, 2014</a>) yet had homology with other <em>E. faecalis</em> genomes on the basis of BLASTn searches. Seven of the 11 unique genes formed a contiguous region suggestive of an operon, which contained two genes with known annotations, <em>mazG</em>, which encodes a nucleoside triphosphate pyrophosphohydrolase involved in survival under nutrient stress, and another that encodes an LPxTG-motif protein cell wall anchor domain protein. Neither ISS_2 (Accession no.: <a href="https://ncbi.nlm.nih.gov/nucleotide/CP046112" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">CP046112</a>) nor ISS_3 (Accession no.: <a href="https://ncbi.nlm.nih.gov/nucleotide/CP046111" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">CP046111</a>) contained individually unique genes. Consistent with this finding, analysis of the core alignment revealed that isolates ISS_2 and ISS_3 differed by only 14 bases. However, these two isolates shared 39 genes that were not present in any of the other strains analyzed, of which 30 were classified as hypothetical. ISS_4 (Accession no.: <a href="https://ncbi.nlm.nih.gov/nucleotide/CP046108-110" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">CP046108-110</a>) had 25 unique genes, 16 of which were located on a plasmid (pTEF2); no unique genes were found on the second (pAD1) plasmid. Based on BLASTn analysis (<a href="#FS2" class="usa-link">Supplementary Figure S2</a>), the majority of genes unique to the ISS strains are closely related to genes present in <em>Enterococcus</em> species, with all non-<em>Enterococcus</em> genera restricted to <em>Lactococcus</em>, <em>Lactobacillus</em>, and, with lower identity and query coverage, <em>Listeria</em> (<a href="#FS3" class="usa-link">Supplementary Figure S3</a>).</p>
<p>Following the designations of <a href="#B39" class="usa-link" aria-describedby="B39">McBride et al. (2007)</a>, the capsule type for each ISS strain was determined. All three known capsule type polymorphisms were detected from the ISS strains. ISS_1 contained the full <em>cps</em> locus (EF2485 to EF2495) and <em>hcp1</em> (EF2484), and similar to MMH594 and V583, was designated CPS type 2. Isolates ISS_2 and ISS_3 contained the <em>cps</em> locus and <em>hcp1</em>, but lacked <em>cpsF</em> (EF0090). In agreement with previous analysis of two additional ST30 isolates (<a href="#B39" class="usa-link" aria-describedby="B39">McBride et al., 2007</a>), these strains were classified as CPS type 5. The strain ISS_4 contained only <em>cpsA</em> (EF0095), <em>cpsB</em> (EF0094), and <em>hcp1</em>, and like the other isolates of CC40 and OG1RF, was classified as CPS type 1 (<a href="#B39" class="usa-link" aria-describedby="B39">McBride et al., 2007</a>).</p></section><section id="S3.SS3"><h3 class="pmc_sec_title">Assessment of Mobile Elements and Defense Systems</h3>
<p>The Comprehensive Antibiotic Resistance Database (CARD) database identified the tetracycline resistance gene, <em>tetW/N/W</em>, in isolates ISS_2, ISS_3, and ISS_4, which was not present in OG1RF (<a href="#B31" class="usa-link" aria-describedby="B31">Jia et al., 2016</a>). The ISS <em>tetW/N/W</em> shared 69.3 (ISS_2 and ISS_3) and 69.1% (ISS_4) sequence homology with the reference sequence (Accession no.: <a href="https://ncbi.nlm.nih.gov/nucleotide/ARO:3004442" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">ARO:3004442</a>). The reference gene has been detected in 2.70% of the available <em>E. faecalis</em> isolates surveyed (<a href="#B31" class="usa-link" aria-describedby="B31">Jia et al., 2016</a>). A standard nucleotide BLAST search confirmed the <em>tetW/N/W</em> genes present in ISS_2 and ISS_3 shared 100% sequence identity to each other, and differed by three SNPs to the <em>tetW/N/W</em> gene present in ISS_4 (<a href="#B1" class="usa-link" aria-describedby="B1">Altschul et al., 1990</a>).</p>
<p>PHASTER identified the presence of intact phage in isolates ISS_2, ISS_3, and ISS_4 (<a href="#TS3" class="usa-link">Supplementary Table S3</a>; <a href="#B2" class="usa-link" aria-describedby="B2">Arndt et al., 2016</a>). ISS_2 and ISS_3 each contained three intact phage nucleotide sequences. The CC40 member ISS_4 contained a single intact phage region. No intact phage were detected in either ISS_1 or OG1RF. Thirty two (82%) of the 39 genes shared only by ISS_2 and ISS_3 were associated with intact or incomplete phage (<a href="#DS1" class="usa-link">Supplementary Data File S1</a>). Plasmid-born genomic elements were only detected in ISS_4 using the tool PlasmidFinder (<a href="#T1" class="usa-link">Table 1</a>; <a href="#B9" class="usa-link" aria-describedby="B9">Carattoli et al., 2014</a>). Two plasmids were determined to share 96.7% sequence identity, and 99% coverage, to the V583 plasmid pTEF2 (accession number: <a href="https://ncbi.nlm.nih.gov/nucleotide/AE016831" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">AE016831</a>; <a href="#B46" class="usa-link" aria-describedby="B46">Paulsen et al., 2003</a>), and 96.0% identity, with 100% coverage, to the pheromone responsive plasmid pAD1 (accession number: <a href="https://ncbi.nlm.nih.gov/nucleotide/L01794" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">L01794</a>; <a href="#B20" class="usa-link" aria-describedby="B20">Ehrenfeld and Clewell, 1987</a>). Notably, evidence for <em>Lactobacillus prophage</em> Lj928 was identified for ISS_2, ISS_3, ISS_4, and V583 (<a href="#TS3" class="usa-link">Supplementary Table S3</a>). Similarly, evidence for <em>Enterococcus phage</em> phiFL4A was identified for ISS_2, ISS_3, ISS_4, and MMH594 (<a href="#TS3" class="usa-link">Supplementary Table S3</a>). Overall, mobile elements explained a majority (65%) of the unique genes identified in the ISS isolates. High confidence CRISPR-Cas systems were detected in isolates ISS_1, ISS_4, and OG1RF (<a href="#TS4" class="usa-link">Supplementary Table S4</a>). CRISPR loci without the Cas genes were detected in the genomes of each of the ISS isolates and OG1RF (<a href="#TS4" class="usa-link">Supplementary Table S4</a>).</p></section><section id="S3.SS4"><h3 class="pmc_sec_title">Genome-Based Assessment of the Potential for Pathogenicity</h3>
<p>The PathogenFinder server scored each of the ISS isolates and the reference strains as potential human pathogens (<a href="#T1" class="usa-link">Table 1</a>; <a href="#B15" class="usa-link" aria-describedby="B15">Cosentino et al., 2013</a>). The commensal strain OG1RF had the lowest pathogen score of 81.6%. With pathogen scores &gt; 84%, all of the ISS strains had a higher assigned probability to be pathogenic than the clinical isolates, MMH594 and V583 (each at 82.9%).</p>
<p>By comparing the genomic content of the isolates to known <em>E. faecalis</em> genes associated with disease, a second predictor of pathogenicity could be compiled (<a href="#T3" class="usa-link">Table 3</a>; <a href="#B33" class="usa-link" aria-describedby="B33">Joensen et al., 2014</a>). The genomes of the clinical isolates contained the highest proportion of predicted virulence factors (18/20), while ISS_2 and ISS_3 contained the least (13/20). ISS_1 and OG1RF both contained 15/20 virulence factors, and the genome ISS_4 was positive for 16/20. The clinical isolates and ISS_4 were the only strains to contain <em>agg</em>, encoding for the <em>E. faecalis</em> aggregation substance involved in biofilm formation, plasmid transfer, and increased virulence in endocarditis models (<a href="#B5" class="usa-link" aria-describedby="B5">Bhatty et al., 2015</a>).</p>
<section class="tw xbox font-sm" id="T3"><h4 class="obj_head">TABLE 3.</h4>
<div class="caption p"><p>The presence, and sequence identity (%), of known virulence factors of <em>Enterococcus faecalis</em> present in the ISS and reference genomes (<a href="#B33" class="usa-link" aria-describedby="B33">Joensen et al., 2014</a>).</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups" cellspacing="5" cellpadding="5">
<thead>
<tr>
<td valign="top" align="left" rowspan="1" colspan="1">Virulence factor</td>
<td valign="top" align="center" colspan="7" rowspan="1">Identity (%)</td>
</tr>
<tr>
<td valign="top" align="center" rowspan="1" colspan="1"></td>
<td valign="top" align="center" colspan="7" rowspan="1"><hr></td>
</tr>
<tr>
<td rowspan="1" colspan="1"></td>
<td valign="top" align="center" rowspan="1" colspan="1">ISS_1</td>
<td valign="top" align="center" rowspan="1" colspan="1">ISS_2</td>
<td valign="top" align="center" rowspan="1" colspan="1">ISS_3</td>
<td valign="top" align="center" rowspan="1" colspan="1">ISS_4</td>
<td valign="top" align="center" rowspan="1" colspan="1">OG1RF</td>
<td valign="top" align="center" rowspan="1" colspan="1">V583</td>
<td valign="top" align="center" rowspan="1" colspan="1">MMH594</td>
</tr>
</thead>
<tbody>
<tr>
<td valign="top" align="left" rowspan="1" colspan="1">ElrA</td>
<td valign="top" align="center" rowspan="1" colspan="1">99.5</td>
<td valign="top" align="center" rowspan="1" colspan="1">99.72</td>
<td valign="top" align="center" rowspan="1" colspan="1">99.72</td>
<td valign="top" align="center" rowspan="1" colspan="1">100</td>
<td valign="top" align="center" rowspan="1" colspan="1">100</td>
<td valign="top" align="center" rowspan="1" colspan="1">100</td>
<td valign="top" align="center" rowspan="1" colspan="1">100</td>
</tr>
<tr>
<td valign="top" align="left" rowspan="1" colspan="1">SrtA</td>
<td valign="top" align="center" rowspan="1" colspan="1">100</td>
<td valign="top" align="center" rowspan="1" colspan="1">100</td>
<td valign="top" align="center" rowspan="1" colspan="1">100</td>
<td valign="top" align="center" rowspan="1" colspan="1">100</td>
<td valign="top" align="center" rowspan="1" colspan="1">100</td>
<td valign="top" align="center" rowspan="1" colspan="1">100</td>
<td valign="top" align="center" rowspan="1" colspan="1">100</td>
</tr>
<tr>
<td valign="top" align="left" rowspan="1" colspan="1"><em>ace</em></td>
<td valign="top" align="center" rowspan="1" colspan="1">98.5</td>
<td valign="top" align="center" rowspan="1" colspan="1">97.71</td>
<td valign="top" align="center" rowspan="1" colspan="1">97.71</td>
<td valign="top" align="center" rowspan="1" colspan="1">96.9</td>
<td valign="top" align="center" rowspan="1" colspan="1">97.5</td>
<td valign="top" align="center" rowspan="1" colspan="1">100</td>
<td rowspan="1" colspan="1"></td>
</tr>
<tr>
<td valign="top" align="left" rowspan="1" colspan="1"><em>agg</em></td>
<td rowspan="1" colspan="1"></td>
<td rowspan="1" colspan="1"></td>
<td rowspan="1" colspan="1"></td>
<td valign="top" align="center" rowspan="1" colspan="1">95.6</td>
<td rowspan="1" colspan="1"></td>
<td valign="top" align="center" rowspan="1" colspan="1">100, 96.8*</td>
<td valign="top" align="center" rowspan="1" colspan="1">100</td>
</tr>
<tr>
<td valign="top" align="left" rowspan="1" colspan="1">cCF10</td>
<td valign="top" align="center" rowspan="1" colspan="1">99.8</td>
<td valign="top" align="center" rowspan="1" colspan="1">99.9, 99.9*</td>
<td valign="top" align="center" rowspan="1" colspan="1">99.9</td>
<td valign="top" align="center" rowspan="1" colspan="1">99.8, 99.8*</td>
<td valign="top" align="center" rowspan="1" colspan="1">100, 100*</td>
<td rowspan="1" colspan="1"></td>
<td rowspan="1" colspan="1"></td>
</tr>
<tr>
<td valign="top" align="left" rowspan="1" colspan="1">cOB1</td>
<td valign="top" align="center" rowspan="1" colspan="1">100</td>
<td valign="top" align="center" rowspan="1" colspan="1">100</td>
<td valign="top" align="center" rowspan="1" colspan="1">100</td>
<td valign="top" align="center" rowspan="1" colspan="1">99.9</td>
<td valign="top" align="center" rowspan="1" colspan="1">100</td>
<td valign="top" align="center" rowspan="1" colspan="1">100</td>
<td valign="top" align="center" rowspan="1" colspan="1">100</td>
</tr>
<tr>
<td valign="top" align="left" rowspan="1" colspan="1"><em>cad</em></td>
<td valign="top" align="center" rowspan="1" colspan="1">99.9</td>
<td valign="top" align="center" rowspan="1" colspan="1">100</td>
<td valign="top" align="center" rowspan="1" colspan="1">100</td>
<td valign="top" align="center" rowspan="1" colspan="1">99.8</td>
<td valign="top" align="center" rowspan="1" colspan="1">100</td>
<td valign="top" align="center" rowspan="1" colspan="1">100</td>
<td valign="top" align="center" rowspan="1" colspan="1">100</td>
</tr>
<tr>
<td valign="top" align="left" rowspan="1" colspan="1"><em>camE</em></td>
<td valign="top" align="center" rowspan="1" colspan="1">100</td>
<td valign="top" align="center" rowspan="1" colspan="1">99.6, 99.6*</td>
<td valign="top" align="center" rowspan="1" colspan="1">99.6</td>
<td valign="top" align="center" rowspan="1" colspan="1">100, 100*</td>
<td valign="top" align="center" rowspan="1" colspan="1">100</td>
<td valign="top" align="center" rowspan="1" colspan="1">100</td>
<td valign="top" align="center" rowspan="1" colspan="1">100</td>
</tr>
<tr>
<td valign="top" align="left" rowspan="1" colspan="1"><em>cylA</em></td>
<td rowspan="1" colspan="1"></td>
<td rowspan="1" colspan="1"></td>
<td rowspan="1" colspan="1"></td>
<td rowspan="1" colspan="1"></td>
<td rowspan="1" colspan="1"></td>
<td rowspan="1" colspan="1"></td>
<td valign="top" align="center" rowspan="1" colspan="1">99.8, 99.8*</td>
</tr>
<tr>
<td valign="top" align="left" rowspan="1" colspan="1"><em>cylL</em></td>
<td rowspan="1" colspan="1"></td>
<td rowspan="1" colspan="1"></td>
<td rowspan="1" colspan="1"></td>
<td rowspan="1" colspan="1"></td>
<td rowspan="1" colspan="1"></td>
<td valign="top" align="center" rowspan="1" colspan="1">100</td>
<td valign="top" align="center" rowspan="1" colspan="1">100</td>
</tr>
<tr>
<td valign="top" align="left" rowspan="1" colspan="1"><em>cylM</em></td>
<td rowspan="1" colspan="1"></td>
<td rowspan="1" colspan="1"></td>
<td rowspan="1" colspan="1"></td>
<td rowspan="1" colspan="1"></td>
<td rowspan="1" colspan="1"></td>
<td valign="top" align="center" rowspan="1" colspan="1">100</td>
<td valign="top" align="center" rowspan="1" colspan="1">100</td>
</tr>
<tr>
<td valign="top" align="left" rowspan="1" colspan="1"><em>ebpA</em></td>
<td valign="top" align="center" rowspan="1" colspan="1">100</td>
<td valign="top" align="center" rowspan="1" colspan="1">100</td>
<td valign="top" align="center" rowspan="1" colspan="1">100</td>
<td valign="top" align="center" rowspan="1" colspan="1">100</td>
<td valign="top" align="center" rowspan="1" colspan="1">100</td>
<td valign="top" align="center" rowspan="1" colspan="1">100</td>
<td valign="top" align="center" rowspan="1" colspan="1">100</td>
</tr>
<tr>
<td valign="top" align="left" rowspan="1" colspan="1"><em>ebpB</em></td>
<td valign="top" align="center" rowspan="1" colspan="1">100</td>
<td valign="top" align="center" rowspan="1" colspan="1">99.8</td>
<td valign="top" align="center" rowspan="1" colspan="1">99.8</td>
<td rowspan="1" colspan="1"></td>
<td rowspan="1" colspan="1"></td>
<td valign="top" align="center" rowspan="1" colspan="1">100</td>
<td valign="top" align="center" rowspan="1" colspan="1">100</td>
</tr>
<tr>
<td valign="top" align="left" rowspan="1" colspan="1"><em>ebpC</em></td>
<td valign="top" align="center" rowspan="1" colspan="1">100</td>
<td rowspan="1" colspan="1"></td>
<td rowspan="1" colspan="1"></td>
<td valign="top" align="center" rowspan="1" colspan="1">100</td>
<td valign="top" align="center" rowspan="1" colspan="1">100</td>
<td valign="top" align="center" rowspan="1" colspan="1">100</td>
<td valign="top" align="center" rowspan="1" colspan="1">100</td>
</tr>
<tr>
<td valign="top" align="left" rowspan="1" colspan="1"><em>efaAfs</em></td>
<td valign="top" align="center" rowspan="1" colspan="1">100</td>
<td valign="top" align="center" rowspan="1" colspan="1">99.9</td>
<td valign="top" align="center" rowspan="1" colspan="1">99.9</td>
<td valign="top" align="center" rowspan="1" colspan="1">100</td>
<td valign="top" align="center" rowspan="1" colspan="1">100</td>
<td valign="top" align="center" rowspan="1" colspan="1">100</td>
<td valign="top" align="center" rowspan="1" colspan="1">100</td>
</tr>
<tr>
<td valign="top" align="left" rowspan="1" colspan="1"><em>fsrB</em></td>
<td valign="top" align="center" rowspan="1" colspan="1">99.9</td>
<td rowspan="1" colspan="1"></td>
<td rowspan="1" colspan="1"></td>
<td valign="top" align="center" rowspan="1" colspan="1">100</td>
<td valign="top" align="center" rowspan="1" colspan="1">100</td>
<td valign="top" align="center" rowspan="1" colspan="1">100</td>
<td valign="top" align="center" rowspan="1" colspan="1">100</td>
</tr>
<tr>
<td valign="top" align="left" rowspan="1" colspan="1"><em>gelE</em></td>
<td valign="top" align="center" rowspan="1" colspan="1">99.4</td>
<td valign="top" align="center" rowspan="1" colspan="1">99.9</td>
<td valign="top" align="center" rowspan="1" colspan="1">99.9</td>
<td valign="top" align="center" rowspan="1" colspan="1">100</td>
<td valign="top" align="center" rowspan="1" colspan="1">100, 100*</td>
<td valign="top" align="center" rowspan="1" colspan="1">100</td>
<td valign="top" align="center" rowspan="1" colspan="1">100</td>
</tr>
<tr>
<td valign="top" align="left" rowspan="1" colspan="1"><em>hylA</em></td>
<td rowspan="1" colspan="1"></td>
<td rowspan="1" colspan="1"></td>
<td rowspan="1" colspan="1"></td>
<td valign="top" align="center" rowspan="1" colspan="1">99.9</td>
<td valign="top" align="center" rowspan="1" colspan="1">100</td>
<td valign="top" align="center" rowspan="1" colspan="1">100</td>
<td valign="top" align="center" rowspan="1" colspan="1">99.7</td>
</tr>
<tr>
<td valign="top" align="left" rowspan="1" colspan="1"><em>hylB</em></td>
<td valign="top" align="center" rowspan="1" colspan="1">99.9</td>
<td valign="top" align="center" rowspan="1" colspan="1">99.5</td>
<td valign="top" align="center" rowspan="1" colspan="1">99.5</td>
<td valign="top" align="center" rowspan="1" colspan="1">99.6</td>
<td valign="top" align="center" rowspan="1" colspan="1">100</td>
<td valign="top" align="center" rowspan="1" colspan="1">100</td>
<td valign="top" align="center" rowspan="1" colspan="1">100</td>
</tr>
<tr>
<td valign="top" align="left" rowspan="1" colspan="1"><em>tpx</em></td>
<td valign="top" align="center" rowspan="1" colspan="1">100</td>
<td valign="top" align="center" rowspan="1" colspan="1">100</td>
<td valign="top" align="center" rowspan="1" colspan="1">100</td>
<td valign="top" align="center" rowspan="1" colspan="1">100</td>
<td valign="top" align="center" rowspan="1" colspan="1">100</td>
<td valign="top" align="center" rowspan="1" colspan="1">100</td>
<td valign="top" align="center" rowspan="1" colspan="1">100</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/T3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="attrib6"><p><em>* Multiple copies were detected.</em></p></div></div></section><p>MMH594 was the only isolate with <em>cylA</em>, one of the genes responsible for the production of cytolysin. Neither OG1RF nor the ISS isolates carried any of the genes in the cytolysin pathway reviewed by <a href="#B14" class="usa-link" aria-describedby="B14">Coburn and Gilmore (2003)</a>.</p></section><section id="S3.SS5"><h3 class="pmc_sec_title">Minimum Inhibitory Concentration of Antibiotics</h3>
<p>Dilution concentration broth assays were used to determine the MIC for various antibiotics. Optical density measurements revealed there were no significant differences in the growth kinetics between the ISS isolates and the reference strains. Strains were first evaluated in Mueller-Hinton broth for comparison to clinical laboratory standards for antibiotic resistance of <em>E. faecalis</em> ATCC<sup>®</sup> 29212 (<a href="#B13" class="usa-link" aria-describedby="B13">CLSI, 2017</a>), and in the commonly used growth media BHI to determine MICs relative to the commensal isolate OG1RF. In MH broth, all strains tested exceeded the resistance levels set forth by ≥ 2-fold for amoxicillin, ampicillin, oxacillin, gentamycin, and kanamycin (<a href="#B13" class="usa-link" aria-describedby="B13">CLSI, 2017</a>). ISS_2, ISS_3, and ISS_4 were also categorized as resistant to tetracycline. When evaluated in BHI, all ISS strains exhibited MICs equivalent to OG1RF with the exceptions of oxacillin and tetracycline (<a href="#T4" class="usa-link">Table 4</a>). All ISS strains showed ≥ 4-fold more resistance to oxacillin. Isolates ISS_2, ISS_3, and ISS_4, each carrying a <em>tetW/N/W</em> gene, displayed an MIC of 64 μg mL<sup>–1</sup>. These isolates were 4-fold more resistant to tetracycline than OG1RF (MIC of 16 μg mL<sup>–1</sup>), and displayed resistance levels equivalent to the clinical isolates, MMH594 and V583. Unlike the clinical isolates, the ISS strains did not exhibit HLGR.</p>
<section class="tw xbox font-sm" id="T4"><h4 class="obj_head">TABLE 4.</h4>
<div class="caption p"><p>Minimum inhibitory concentration of antibiotics evaluated in BHI.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups" cellspacing="5" cellpadding="5">
<thead>
<tr>
<td valign="top" align="left" rowspan="1" colspan="1">Isolate</td>
<td valign="top" align="center" colspan="9" rowspan="1">Antibiotic MIC (μg mL<sup>–1</sup>)</td>
</tr>
<tr>
<td valign="top" align="center" rowspan="1" colspan="1"></td>
<td valign="top" align="center" colspan="9" rowspan="1"><hr></td>
</tr>
<tr>
<td rowspan="1" colspan="1"></td>
<td valign="top" align="center" rowspan="1" colspan="1">Penicillin</td>
<td valign="top" align="center" rowspan="1" colspan="1">Amoxicillin</td>
<td valign="top" align="center" rowspan="1" colspan="1">Ampicillin</td>
<td valign="top" align="center" rowspan="1" colspan="1">Oxacillin</td>
<td valign="top" align="center" rowspan="1" colspan="1">Erythromycin</td>
<td valign="top" align="center" rowspan="1" colspan="1">Tetracycline</td>
<td valign="top" align="center" rowspan="1" colspan="1">Gentamycin</td>
<td valign="top" align="center" rowspan="1" colspan="1">Streptomycin</td>
<td valign="top" align="center" rowspan="1" colspan="1">Kanamycin</td>
</tr>
</thead>
<tbody>
<tr>
<td valign="top" align="left" rowspan="1" colspan="1">ISS-1</td>
<td valign="top" align="center" rowspan="1" colspan="1">10.0</td>
<td valign="top" align="center" rowspan="1" colspan="1">&gt; 10.0</td>
<td valign="top" align="center" rowspan="1" colspan="1">2.50</td>
<td valign="top" align="center" rowspan="1" colspan="1">256</td>
<td valign="top" align="center" rowspan="1" colspan="1">1.56</td>
<td valign="top" align="center" rowspan="1" colspan="1">16.0</td>
<td valign="top" align="center" rowspan="1" colspan="1">125</td>
<td valign="top" align="center" rowspan="1" colspan="1">500</td>
<td valign="top" align="center" rowspan="1" colspan="1">500</td>
</tr>
<tr>
<td valign="top" align="left" rowspan="1" colspan="1">ISS-2</td>
<td valign="top" align="center" rowspan="1" colspan="1">5.00</td>
<td valign="top" align="center" rowspan="1" colspan="1">&gt; 10.0</td>
<td valign="top" align="center" rowspan="1" colspan="1">2.50</td>
<td valign="top" align="center" rowspan="1" colspan="1">128</td>
<td valign="top" align="center" rowspan="1" colspan="1">1.56</td>
<td valign="top" align="center" rowspan="1" colspan="1">64.0</td>
<td valign="top" align="center" rowspan="1" colspan="1">125</td>
<td valign="top" align="center" rowspan="1" colspan="1">500</td>
<td valign="top" align="center" rowspan="1" colspan="1">500</td>
</tr>
<tr>
<td valign="top" align="left" rowspan="1" colspan="1">ISS-3</td>
<td valign="top" align="center" rowspan="1" colspan="1">5.00</td>
<td valign="top" align="center" rowspan="1" colspan="1">&gt; 10.0</td>
<td valign="top" align="center" rowspan="1" colspan="1">2.50</td>
<td valign="top" align="center" rowspan="1" colspan="1">128</td>
<td valign="top" align="center" rowspan="1" colspan="1">1.56</td>
<td valign="top" align="center" rowspan="1" colspan="1">64.0</td>
<td valign="top" align="center" rowspan="1" colspan="1">125</td>
<td valign="top" align="center" rowspan="1" colspan="1">500</td>
<td valign="top" align="center" rowspan="1" colspan="1">500</td>
</tr>
<tr>
<td valign="top" align="left" rowspan="1" colspan="1">ISS-4</td>
<td valign="top" align="center" rowspan="1" colspan="1">10.0</td>
<td valign="top" align="center" rowspan="1" colspan="1">&gt; 10.0</td>
<td valign="top" align="center" rowspan="1" colspan="1">2.50</td>
<td valign="top" align="center" rowspan="1" colspan="1">128</td>
<td valign="top" align="center" rowspan="1" colspan="1">1.56</td>
<td valign="top" align="center" rowspan="1" colspan="1">&gt; 64.0</td>
<td valign="top" align="center" rowspan="1" colspan="1">62.5</td>
<td valign="top" align="center" rowspan="1" colspan="1">500</td>
<td valign="top" align="center" rowspan="1" colspan="1">250</td>
</tr>
<tr>
<td valign="top" align="left" rowspan="1" colspan="1">OG1RF</td>
<td valign="top" align="center" rowspan="1" colspan="1">5.00</td>
<td valign="top" align="center" rowspan="1" colspan="1">&gt; 10.0</td>
<td valign="top" align="center" rowspan="1" colspan="1">2.50</td>
<td valign="top" align="center" rowspan="1" colspan="1">32.0</td>
<td valign="top" align="center" rowspan="1" colspan="1">1.56</td>
<td valign="top" align="center" rowspan="1" colspan="1">16.0</td>
<td valign="top" align="center" rowspan="1" colspan="1">125</td>
<td valign="top" align="center" rowspan="1" colspan="1">500</td>
<td valign="top" align="center" rowspan="1" colspan="1">500</td>
</tr>
<tr>
<td valign="top" align="left" rowspan="1" colspan="1">MMH594</td>
<td valign="top" align="center" rowspan="1" colspan="1">10.0</td>
<td valign="top" align="center" rowspan="1" colspan="1">&gt; 10.0</td>
<td valign="top" align="center" rowspan="1" colspan="1">5.00</td>
<td valign="top" align="center" rowspan="1" colspan="1">128</td>
<td valign="top" align="center" rowspan="1" colspan="1">&gt; 50.0</td>
<td valign="top" align="center" rowspan="1" colspan="1">64.0</td>
<td valign="top" align="center" rowspan="1" colspan="1">&gt; 2.00 × 10<sup>3</sup>
</td>
<td valign="top" align="center" rowspan="1" colspan="1">1.00 × 10<sup>3</sup>
</td>
<td valign="top" align="center" rowspan="1" colspan="1">&gt; 2.00 × 10<sup>3</sup>
</td>
</tr>
<tr>
<td valign="top" align="left" rowspan="1" colspan="1">V583</td>
<td valign="top" align="center" rowspan="1" colspan="1">2.50</td>
<td valign="top" align="center" rowspan="1" colspan="1">10.0</td>
<td valign="top" align="center" rowspan="1" colspan="1">2.50</td>
<td valign="top" align="center" rowspan="1" colspan="1">256</td>
<td valign="top" align="center" rowspan="1" colspan="1">&gt; 50.0</td>
<td valign="top" align="center" rowspan="1" colspan="1">64.0</td>
<td valign="top" align="center" rowspan="1" colspan="1">&gt; 2.00 × 10<sup>3</sup>
</td>
<td valign="top" align="center" rowspan="1" colspan="1">500</td>
<td valign="top" align="center" rowspan="1" colspan="1">&gt; 2.00 × 10<sup>3</sup>
</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/T4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="S3.SS6"><h3 class="pmc_sec_title">Desiccation Tolerance of <em>E. faecalis</em> Strains</h3>
<p>During the initial 24 h dry down period, the RH in the desiccation chamber spiked to 56%. For the remainder of the experiment, the RH was maintained between 30-40% by adding additional Drierite<sup>®</sup>. The concentration of cells surviving the drying process (<em>N</em><sub>0</sub>) ranged from 1.2 (± 0.044) x 10<sup>10</sup> to 5.5 (± 0.091) x 10<sup>7</sup> CFU for ISS-1 and OG1RF, respectively. The average number of survivors (<em>N</em>) after three weeks of desiccation ranged from 0.11 (± 0.024) to 8.5 (± 0.79) x 10<sup>8</sup> CFU for OG1RF and MMH594, respectively. OG1RF demonstrated the greatest loss in viability during dry down with a 2.1-log reduction, and ISS-2 was the only isolate that did not experience a significant loss in viability between <em>t</em> = 0 and <em>t</em> = 3 d (<em>p</em> = 0.11). There was no significant difference in <em>N/N<sub>0</sub></em> at any of the timepoints evaluated (<em>p</em> = 0.98; <a href="#F3" class="usa-link">Figure 3</a>). When the variances of <em>N/N<sub>0</sub></em> at t = 21 d for each isolate were determined to be significantly different from each other (<em>F</em> = 58, <em>p</em> &lt; 0.001), a non-parametric comparison was performed with OG1RF chosen as the control using the Steel method. There was no significant difference in <em>N/N<sub>0</sub></em> at <em>t</em> = 21 d among the isolates tested when compared to OG1RF (<a href="#F3" class="usa-link">Figure 3</a>).</p>
<figure class="fig xbox font-sm" id="F3"><h4 class="obj_head">FIGURE 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=7829349_fmicb-11-515319-g003.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/98bf/7829349/b1f793e7d9c5/fmicb-11-515319-g003.jpg" loading="lazy" height="386" width="738" alt="FIGURE 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/F3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Desiccation survival of <em>Enterococcus faecalis</em> strains from the ISS and the control strains, OG1RF and MMH594. The data are presented as the average (± SD) of triplicate measurements. Some error bars have been obscured by the data points.</p></figcaption></figure></section><section id="S3.SS7"><h3 class="pmc_sec_title">Pathogenicity of Isolates in a <em>C. elegans</em> Model</h3>
<p>The survival data (<a href="#F4" class="usa-link">Figure 4</a>) were determined to follow the normal distribution using the Shapiro-Wilk test (<a href="#B25" class="usa-link" aria-describedby="B25">Han et al., 2016</a>). With the exception of ISS_1, all data are reported as the mean (± SE) of three replicates. For ISS_1, one sample became contaminated with fungus on day 9, and all subsequent data are reported as <em>N</em> = 2. <em>C. elegans</em> was least susceptible to strains ISS_2, ISS_3, and V583 with mean survival times of 13 (± 0.53), 13 (± 0.59), and 11 (± 0.91) days, respectively (<a href="#T2" class="usa-link">Table 2</a>). While the time for 50% of the worm population to die (LT<sub>50</sub>; <a href="#B23" class="usa-link" aria-describedby="B23">Garsin et al., 2001</a>) for these isolates ranged from 9 days (V583) to 14 (ISS_3), the total lifespan of worms challenged with these isolates were not significantly different from each other. For isolates ISS_2 and ISS_3, all worms were deceased by day 18 and 17, respectively. While 90% of worms fed V583 were dead by day 18, some survivors persisted until day 24. Additionally, the lifespan of worms infected with ISS_3 was also statistically similar to those challenged with ISS_4. With a 31% decrease in the mean lifespan, worms infected with ISS_1 were significantly more susceptible than isolates fed ISS_2 or ISS_3 (<em>p</em> &lt; 0.001). Based on the survival curves of worms fed ISS_1, ISS_4, OG1RF, MMH594, and V583 there were no significant differences in the pathogenicity profiles for these isolates (<em>p</em> &gt; 0.05).</p>
<figure class="fig xbox font-sm" id="F4"><h4 class="obj_head">FIGURE 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=7829349_fmicb-11-515319-g004.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/98bf/7829349/292c1abc0fef/fmicb-11-515319-g004.jpg" loading="lazy" height="382" width="738" alt="FIGURE 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/F4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The average survival of <em>Caenorhabditis elegans</em> exposed to ISS and control strains of <em>Enterococcus faecalis.</em></p></figcaption></figure></section></section><section id="S4"><h2 class="pmc_sec_title">Discussion</h2>
<p>While multiple studies have detected the presence of a wide range of opportunistic human pathogens onboard the ISS, the consensus was that additional data was needed to determine if these microorganisms posed an actual threat to crew health (<a href="#B51" class="usa-link" aria-describedby="B51">Schiwon et al., 2013</a>; <a href="#B4" class="usa-link" aria-describedby="B4">Be et al., 2017</a>; <a href="#B35" class="usa-link" aria-describedby="B35">Lang et al., 2017</a>; <a href="#B56" class="usa-link" aria-describedby="B56">Singh et al., 2018a</a>,<a href="#B57" class="usa-link" aria-describedby="B57">b</a>; <a href="#B43" class="usa-link" aria-describedby="B43">Mora et al., 2019</a>; <a href="#B55" class="usa-link" aria-describedby="B55">Sielaff et al., 2019</a>,<a href="#B58" class="usa-link" aria-describedby="B58">Sobisch et al., 2019</a>; <a href="#B63" class="usa-link" aria-describedby="B63">Urbaniak et al., 2019</a>). Here, we expanded upon previous genomic analyses in an effort to provide a framework for the determination of the pathogenic potential of <em>E. faecalis</em> isolates. For the comparative analyses, the commensal type strain OG1RF and the clinical isolates, MMH594 and V583, were chosen because of their extensive genotypic (<a href="#B54" class="usa-link" aria-describedby="B54">Shankar et al., 2002</a>; <a href="#B46" class="usa-link" aria-describedby="B46">Paulsen et al., 2003</a>; <a href="#B39" class="usa-link" aria-describedby="B39">McBride et al., 2007</a>; <a href="#B7" class="usa-link" aria-describedby="B7">Bourgogne et al., 2008</a>) and phenotypic characterization, specifically in regards to <em>C. elegans</em> pathogenesis (<a href="#B23" class="usa-link" aria-describedby="B23">Garsin et al., 2001</a>; <a href="#B71" class="usa-link" aria-describedby="B71">Yuen and Ausubel, 2018</a>). In addition, the <em>E. faecalis</em> genomes of strains 5952 and JH-1 were chosen as representative ST strains of the ISS isolates ISS_2 and ISS_3, and ISS_4, respectively.</p>
<p>While the amount of information that can be gleaned from a complete genome is large, the current methodologies (<a href="#B56" class="usa-link" aria-describedby="B56">Singh et al., 2018a</a>) are insufficient for the accurate assessment of pathogenicity for <em>E. faecalis</em> strains. The scores from the tool, PathogenFinder (<a href="#B15" class="usa-link" aria-describedby="B15">Cosentino et al., 2013</a>), predicted each of the ISS isolates as possessing &gt; 84% probability of being pathogenic to humans, placing them above known clinical isolates MMH594 and V583 (<a href="#T1" class="usa-link">Table 1</a>). Another tool related to scoring pathogenicity, VirFinder, identified twenty virulence factors among the ISS strains, but fifteen were present in the commensal isolate OG1RF as well (<a href="#T3" class="usa-link">Table 3</a>). We cautiously interpret this to mean that ISS isolates do share some of the potentially virulence-related properties found in clinical isolates, but many of these are also shared by prototype commensal strains of <em>E. faecalis</em>. Further validation for these predictive algorithms applied to enterococci is needed in order to make more accurate predictions of which strains could impact human health. Predictions from PathogenFinder or its equivalent can be expected to improve if additional training data at the genus or species level is incorporated.</p>
<p>Because of the difficulty in interpreting potential pathogenicity of the ISS strains based on genome content alone, we tested them directly in the <em>C. elegans</em> infection model (<a href="#F4" class="usa-link">Figure 4</a>). Previous studies identified parallels in the pathogenicity profiles of various enterococcal lineages when tested in <em>C. elegans</em> (<a href="#B23" class="usa-link" aria-describedby="B23">Garsin et al., 2001</a>) and in mice (<a href="#B7" class="usa-link" aria-describedby="B7">Bourgogne et al., 2008</a>). In contrast to that report (<a href="#B23" class="usa-link" aria-describedby="B23">Garsin et al., 2001</a>), we did not detect significant difference in <em>C. elegans</em> survival when exposed to commensal OG1RF, or pathogenic V583 and the cytolytic strain MMH594. <em>E. faecalis</em> proliferate in the intestine of <em>C. elegans</em> (<a href="#B23" class="usa-link" aria-describedby="B23">Garsin et al., 2001</a>) ultimately occluding it (<a href="#B71" class="usa-link" aria-describedby="B71">Yuen and Ausubel, 2018</a>). As a result, <em>C. elegans</em> infected with <em>E. faecalis</em> reach the LT<sub>50</sub> 33% faster than worms fed a normal diet of <em>E. coli</em> OP50 (<a href="#B23" class="usa-link" aria-describedby="B23">Garsin et al., 2001</a>). None of the <em>E. faecalis</em> strains tested here, including ISS strains, exhibited enhanced <em>C. elegans</em> killing (<a href="#F4" class="usa-link">Figure 4</a>). Closely related strains ISS_2 and 3, despite possessing a high computed pathogenicity score, were found to be least able to kill <em>C. elegans</em> (<a href="#F4" class="usa-link">Figure 4</a>).</p>
<p>Additionally, there are currently no validated methods to predict phenotypic responses of enterococci to environmental stressors (<em>e.g</em>., desiccation, starvation, increased cosmic radiation) based on their genomic content alone. Because each of the ISS isolates were recovered from air and surface samples, we examined them specifically for the ability to survive desiccation at RH relevant to the clinical environment (<a href="#B3" class="usa-link" aria-describedby="B3">ASHRAE, 2013</a>). In general, members of the genus <em>Enterococcus</em> are more desiccation tolerant than closely related microorganisms (<a href="#B37" class="usa-link" aria-describedby="B37">Lebreton et al., 2017</a>). <a href="#B6" class="usa-link" aria-describedby="B6">Blaustein et al. (2019)</a> demonstrated that the spaceflight environment onboard the ISS did not select for “extremo-tolerant” strains of <em>B. cereus</em> and <em>S. aureus</em>. Likewise, we found no significant differences in desiccation tolerance between the ISS <em>E. faecalis</em> isolates and their representative type strains (<a href="#F3" class="usa-link">Figure 3</a>). This indicates that their environmental survival in the ISS was attributable to the previously described intrinsically rugged nature of E. <em>faecalis</em> (<a href="#B37" class="usa-link" aria-describedby="B37">Lebreton et al., 2017</a>). <a href="#B61" class="usa-link" aria-describedby="B61">Tirumalai et al. (2018)</a> compared the genomes of the highly resistant <em>B. safensis</em> FO-36b and <em>B. pumilus</em> SAFR-032 to over 60 reference strains, and also failed to find genomic signatures for their increased environmental resistances (<em>e.g.</em>, desiccation, starvation, increased ultraviolet radiation). Rather, it is more likely the regulation of a suite of genes that confers increased resistance as a generalized stress response. In agreement with <a href="#B6" class="usa-link" aria-describedby="B6">Blaustein et al. (2019)</a> we conclude that the interior space environment (<em>e.g.</em>, microgravity, increased cosmic radiation) selects for intrinsically rugged species of bacteria, such as <em>E. faecalis</em>, but as of yet, there is no evidence suggesting selection for enhanced environmental persistence.</p>
<p>In agreement with previous observations (<a href="#B51" class="usa-link" aria-describedby="B51">Schiwon et al., 2013</a>; <a href="#B58" class="usa-link" aria-describedby="B58">Sobisch et al., 2019</a>), the <em>E. faecalis</em> isolates examined here were resistant to multiple antibiotics. While the detection of multidrug resistant bacteria may sound alarming, much of that resistance is intrinsic to all enterococci, and other genetic signatures indicate that these are largely commensal strains (<a href="#T3" class="usa-link">Table 3</a>). However, resistance to tetracycline displayed by ISS_2, ISS_3, and ISS_4 was found to be mediated by an acquired <em>tetW/N/W</em> gene, which confers an MIC equivalent to that of clinical isolates (<a href="#T4" class="usa-link">Table 4</a>). In addition to harboring genes that make combating infections difficult, <em>E. faecalis</em> strains possess the ability to spread antibiotic resistance to genes to other bacterial species, including the commonly isolated ISS inhabitant <em>S. aureus</em> (<a href="#B58" class="usa-link" aria-describedby="B58">Sobisch et al., 2019</a>). Although there have been no reports of VRE isolated from onboard the ISS at the time of this publication, it is important to note that both vancomycin resistance genes and <em>E. faecalis</em> genomic signatures were detected concurrently from the surface of the dining table during expedition 43 (<a href="#B57" class="usa-link" aria-describedby="B57">Singh et al., 2018b</a>). During the same mission, <em>E. faecalis</em> was also ranked the second most abundant BSL-2 microorganism in the metagenomic analysis of the entire campaign (<a href="#B57" class="usa-link" aria-describedby="B57">Singh et al., 2018b</a>). In fact, <a href="#B43" class="usa-link" aria-describedby="B43">Mora et al. (2019)</a> determined <em>Enterococcus</em> to be one of the ten indicator genera that distinguished the ISS microbiome from the human and indoor samples of the Home Microbiome Project (<a href="#B43" class="usa-link" aria-describedby="B43">Mora et al., 2019</a>).</p>
<p>There were several limitations to the data presented above. The desiccation survival experiments were terminated before any significant decrease in viability was observed. Extending the experiment until no surviving cells remained may have revealed a difference in the survival curves among the strains and provide insight to the length of time <em>E. faecalis</em> can remain viable. The lack of experimentally validated computational tools for <em>E. faecalis</em> led to conflicting results when comparing the genomes known clinical isolates to commensal strains. In addition, <em>C. elegans</em> serves as a rudimentary model system for human infection. However, the challenges presented here provide the foundation for future investigations of <em>E. faecalis</em>, on Earth or in space.</p>
<p>As members of the core gut microbiome, <em>E. faecalis</em> will continue to be transported to the ISS and the ability to detect pathogenic strains will be beneficial for crew health. Unlike most environmental bacteria (<a href="#B59" class="usa-link" aria-describedby="B59">Staley and Konopka, 1985</a>), <em>E. faecalis</em> strains are routinely recovered on culture media. Based on the antibiotic resistance profiles presented here, incorporating the above high-level aminoglycosides to the routine onboard culturing activities could serve as the first step toward distinguishing pathogenic from commensal strains of <em>E. faecalis</em>. Because of their commonness as survivors on abiotic surfaces, their potential ability to infect, as well as the ability of <em>E. faecalis</em> to readily acquire and spread mobile elements conferring antibiotic resistance, continued microbial monitoring and pathogen identification will be important.</p>
<p>Current microbial monitoring of ISS air and surface sampling occurs once a month for the first 90 days of a mission, and decreases to one sample event for every 90 days thereafter (<a href="#B29" class="usa-link" aria-describedby="B29">ISS-MORD, 2013</a>). Because these microbial monitoring procedures are culture based, there is an inherent bias toward the small fraction of bacterial species that will form a colony on a plate (<a href="#B59" class="usa-link" aria-describedby="B59">Staley and Konopka, 1985</a>). Assessing the total microbial burden, and viable fraction, by combining genomic and culture-based analyses provides a more accurate representation of the ISS microbiome (<a href="#B4" class="usa-link" aria-describedby="B4">Be et al., 2017</a>; <a href="#B43" class="usa-link" aria-describedby="B43">Mora et al., 2019</a>; <a href="#B55" class="usa-link" aria-describedby="B55">Sielaff et al., 2019</a>). However, it is still unclear which, if any, of these microorganisms are relevant to crew health (<a href="#B51" class="usa-link" aria-describedby="B51">Schiwon et al., 2013</a>; <a href="#B4" class="usa-link" aria-describedby="B4">Be et al., 2017</a>; <a href="#B35" class="usa-link" aria-describedby="B35">Lang et al., 2017</a>; <a href="#B56" class="usa-link" aria-describedby="B56">Singh et al., 2018a</a>,<a href="#B57" class="usa-link" aria-describedby="B57">b</a>; <a href="#B43" class="usa-link" aria-describedby="B43">Mora et al., 2019</a>; <a href="#B55" class="usa-link" aria-describedby="B55">Sielaff et al., 2019</a>; <a href="#B58" class="usa-link" aria-describedby="B58">Sobisch et al., 2019</a>; <a href="#B63" class="usa-link" aria-describedby="B63">Urbaniak et al., 2019</a>).</p>
<p>The need for expedient identification of pathogenic microorganisms and their antibiotic resistance profiles is not a challenge unique to the spaceflight environment. Recent advances in rapid diagnostics have helped alleviate the clinical reliance on culture based techniques to identify infectious agents, yet these resources are not always widely available (<a href="#B38" class="usa-link" aria-describedby="B38">Maurer et al., 2017</a>). Fortunately, this is not the case for the ISS where the crew have access to Wetlab-2 Cepheid© SmartCycyler that can be used for quantitative PCR (qPCR) analysis of air, surface, water, and clinical samples<sup><a href="#footnote3" class="usa-link">3</a></sup>. Although it is currently configured to detect only a limited number of microbial pathogens, the RAZOR<sup>TM</sup> EX BioDetection System (Idaho Technology Inc.) is currently being developed for in-flight microbial monitoring and could be modified for a wider range of microbial targets. Another promising technology for pathogen detection is the ONT MinION sequencing device employed above. <a href="#B11" class="usa-link" aria-describedby="B11">Castro-Wallace et al. (2017)</a> have previously validated its efficacy in the spacecraft environment. The MinION is currently capable of identifying bacterial species in near real time from metagenomic data (<a href="#B50" class="usa-link" aria-describedby="B50">Sanderson et al., 2018</a>) or isolated samples (<a href="#B22" class="usa-link" aria-describedby="B22">Gargis et al., 2019</a>) and has been used for the first ever off-Earth identification of bacteria cultured and sequenced in flight (<a href="#B8" class="usa-link" aria-describedby="B8">Burton et al., 2020</a>).</p>
<p>Due to the inability for the existing tools to distinguish pathogenic from commensal strains of <em>E. faecalis</em>, we make the following recommendations: Rather than continuing to wait for sample return and analysis using the current unvalidated databases, we suggest performing an initial screen for HLGR. Then, any isolates exhibiting HLGR should be sequenced using the MinION and evaluated for the presence of the type 2 capsule (<a href="#B39" class="usa-link" aria-describedby="B39">McBride et al., 2007</a>), the vancomycin resistance operon observed in V583 (<a href="#B46" class="usa-link" aria-describedby="B46">Paulsen et al., 2003</a>), components of the pathogenicity island (<em>e.g</em>., cytolysin production loci and the enterococcal surface protein, Esp), and the absence of functional CRISPR-Cas arrays (<a href="#B64" class="usa-link" aria-describedby="B64">Van Tyne et al., 2019</a>). This information would then provide near real-time information for the crew regarding the need for additional decontamination procedures. These advancements would not be limited to <em>E. faecalis</em> isolates, however more accurate genomic targets for additional pathogen detection will need to be experimentally validated (<a href="#B12" class="usa-link" aria-describedby="B12">Chiu and Miller, 2019</a>). With the development of such targets, multiple pathogens (including bacteria, fungi, and viruses) can quickly be identified within a single sample (<a href="#B42" class="usa-link" aria-describedby="B42">Miller et al., 2019</a>).</p></section><section id="S5"><h2 class="pmc_sec_title">Data Availability Statement</h2>
<p>The datasets generated for this study can be found in the NCBI BioProject website (<a href="https://www.ncbi.nlm.nih.gov/bioproject" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/bioproject</a>) using the BioProject ID: <a href="https://ncbi.nlm.nih.gov/nucleotide/PRJNA587161" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PRJNA587161</a> and the accession numbers <a href="https://ncbi.nlm.nih.gov/nucleotide/SAMN13182395" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">SAMN13182395</a>–<a href="https://ncbi.nlm.nih.gov/nucleotide/SAMN13182398" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">SAMN13182398</a>.</p></section><section id="S6"><h2 class="pmc_sec_title">Author Contributions</h2>
<p>NB designed the experiments, performed the research, analyzed the data, and wrote the manuscript. CC assembled the genomes and advised on bioinformatic analysis and experimental design. MG advised on experimental design. FL advised on bioinformatic analysis and experimental design. GR and MZ advised and reviewed the manuscript. All authors contributed to the manuscript.</p></section><section id="conf1"><h2 class="pmc_sec_title">Conflict of Interest</h2>
<p>The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.</p></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>The authors would like to thank members of the Ruvkun, Ausubel, and Gilmore labs for their guidance during the experimental design. We also thank M. Ott for his input regarding the ISS samples.</p></section><section id="fn-group1" class="fn-group"><div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn4"><p><strong>Funding.</strong> NB was supported by a NASA Space Biology Fellowship, award 80NSSC17K0688. CC was supported by NASA award NNX15AF85G.</p></div></div></section><section id="fn-group2" class="fn-group"><div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="footnote1">
<sup>1</sup><p class="display-inline"><a href="https://lab.loman.net/protocols/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://lab.loman.net/protocols/</a></p>
</div>
<div class="fn p" id="footnote2">
<sup>2</sup><p class="display-inline"><a href="https://github.com/rrwick/Porechop" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/rrwick/Porechop</a></p>
</div>
<div class="fn p" id="footnote3">
<sup>3</sup><p class="display-inline"><a href="https://www.nasa.gov/spacebio/microbiology/hardware" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.nasa.gov/spacebio/microbiology/hardware</a></p>
</div>
</div></section><section id="S9"><h2 class="pmc_sec_title">Supplementary Material</h2>
<p>The Supplementary Material for this article can be found online at: <a href="https://www.frontiersin.org/articles/10.3389/fmicb.2020.515319/full#supplementary-material" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.frontiersin.org/articles/10.3389/fmicb.2020.515319/full#supplementary-material</a></p>
<section class="sm xbox font-sm" id="FS1"><div class="caption p">
<span>Supplementary Figure 1</span><p>Gene membership of the 51 <em>Enterococcus faecalis</em> isolate genomes analyzed. Counts and categories as estimated by Roary analysis of presence/absence as described in the methods.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/7829349/bin/Data_Sheet_2.PDF" data-ga-action="click_feat_suppl" class="usa-link">Click here for additional data file.</a><sup> (4MB, PDF) </sup>
</div></div></section><section class="sm xbox font-sm" id="FS2"><div class="caption p">
<span>Supplementary Figure 2</span><p>Sequence similarity of <em>Enterococcus faecalis</em> genes unique to ISS_1, ISS_2/ISS_3, and ISS_4. Query coverage and percentage sequence identity from BLASTn searches across all isolates (upper left) and for each isolate, respectively. Because ISS_2 and ISS_3 have no individually unique genes, and differ in their core gene alignment by only 14 bases, but have genes that are not represented in any of the other 49 genomes studied, ISS_3 was excluded from this analysis. <a href="#DS1" class="usa-link">Supplementary Data File 1</a> contains details of each BLASTn hit.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/7829349/bin/Data_Sheet_2.PDF" data-ga-action="click_feat_suppl" class="usa-link">Click here for additional data file.</a><sup> (4MB, PDF) </sup>
</div></div></section><section class="sm xbox font-sm" id="FS3"><div class="caption p">
<span>Supplementary Figure 3</span><p>Sequence similarity of <em>Enterococcus faecalis</em> genes unique to ISS_1, ISS_2/ISS_3, and ISS_4 segmented by the genus of the hit subject. Query coverage and percentage sequence identity from BLASTn searches are as in <a href="#FS2" class="usa-link">Supplementary Figure S2</a> (upper left) after eliminating all hits associated with <em>Enterococcus</em> (upper left) and further segmenting by genus (there were only three remaining genera).</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/7829349/bin/Data_Sheet_2.PDF" data-ga-action="click_feat_suppl" class="usa-link">Click here for additional data file.</a><sup> (4MB, PDF) </sup>
</div></div></section><section class="sm xbox font-sm" id="TS1"><div class="caption p">
<span>Supplementary Table 1</span><p>Illumina HiSeq short read coverage estimates.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/7829349/bin/Data_Sheet_2.PDF" data-ga-action="click_feat_suppl" class="usa-link">Click here for additional data file.</a><sup> (4MB, PDF) </sup>
</div></div></section><section class="sm xbox font-sm" id="TS2"><div class="caption p">
<span>Supplementary Table 2</span><p>Nanopore long read coverage estimates.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/7829349/bin/Data_Sheet_2.PDF" data-ga-action="click_feat_suppl" class="usa-link">Click here for additional data file.</a><sup> (4MB, PDF) </sup>
</div></div></section><section class="sm xbox font-sm" id="TS3"><div class="caption p">
<span>Supplementary Table 3</span><p>Predicted intact phage content as determined by PHASTER for ISS and control <em>E. faecalis</em> isolates (<a href="#B2" class="usa-link" aria-describedby="B2">Arndt et al., 2016</a>).</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/7829349/bin/Data_Sheet_2.PDF" data-ga-action="click_feat_suppl" class="usa-link">Click here for additional data file.</a><sup> (4MB, PDF) </sup>
</div></div></section><section class="sm xbox font-sm" id="TS4"><div class="caption p">
<span>Supplementary Table 4</span><p>CRISPR-Cas system detection in the genomes of ISS and reference strains (<a href="#B16" class="usa-link" aria-describedby="B16">Couvin et al., 2018</a>).</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/7829349/bin/Data_Sheet_2.PDF" data-ga-action="click_feat_suppl" class="usa-link">Click here for additional data file.</a><sup> (4MB, PDF) </sup>
</div></div></section><section class="sm xbox font-sm" id="DS1"><div class="caption p">
<span>Supplementary Data File 1</span><p>Excel file listing all genomes used in the project, the detected presence/absence of each annotated gene for all genomes, and BLASTn hits for genes unique to the ISS isolates.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/7829349/bin/Data_Sheet_1.XLSX" data-ga-action="click_feat_suppl" class="usa-link">Click here for additional data file.</a><sup> (2.4MB, XLSX) </sup>
</div></div></section></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ol class="ref-list font-sm">
<li id="B1">
<cite>Altschul S. F., Gish W., Miller W., Myers E. W., Lipman D. J. (1990). Basic local alignment search tool.
<em>J. Mol. Biol.</em>
215
403–410.</cite> [<a href="https://doi.org/10.1016/S0022-2836(05)80360-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2231712/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Mol.%20Biol.&amp;title=Basic%20local%20alignment%20search%20tool.&amp;author=S.%20F.%20Altschul&amp;author=W.%20Gish&amp;author=W.%20Miller&amp;author=E.%20W.%20Myers&amp;author=D.%20J.%20Lipman&amp;volume=215&amp;publication_year=1990&amp;pages=403-410&amp;pmid=2231712&amp;doi=10.1016/S0022-2836(05)80360-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B2">
<cite>Arndt D., Grant J. R., Marcu A., Sajed T., Pon A., Liang Y., et al.  (2016). PHASTER: a better, faster version of the PHAST phage search tool.
<em>Nucleic Acids Res.</em>
44
W16–W21.</cite> [<a href="https://doi.org/10.1093/nar/gkw387" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4987931/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27141966/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=PHASTER:%20a%20better,%20faster%20version%20of%20the%20PHAST%20phage%20search%20tool.&amp;author=D.%20Arndt&amp;author=J.%20R.%20Grant&amp;author=A.%20Marcu&amp;author=T.%20Sajed&amp;author=A.%20Pon&amp;volume=44&amp;publication_year=2016&amp;pages=W16-W21&amp;pmid=27141966&amp;doi=10.1093/nar/gkw387&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B3">
<cite>ASHRAE (2013). <em>ANSI/ASHRAE Standard 170-2013, Ventilation of Health Care Facilities.</em>
Atlanta, GA: American Society of Heating, Inc.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=ANSI/ASHRAE%20Standard%20170-2013,%20Ventilation%20of%20Health%20Care%20Facilities.&amp;publication_year=2013&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B4">
<cite>Be N. A., Avila-Herrera A., Allen J. E., Singh N., Checinska Sielaff A., Jaing C., et al.  (2017). Whole metagenome profiles of particulates collected from the International Space Station.
<em>Microbiome</em>
5:81.</cite> [<a href="https://doi.org/10.1186/s40168-017-0292-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5514531/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28716113/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microbiome&amp;title=Whole%20metagenome%20profiles%20of%20particulates%20collected%20from%20the%20International%20Space%20Station.&amp;author=N.%20A.%20Be&amp;author=A.%20Avila-Herrera&amp;author=J.%20E.%20Allen&amp;author=N.%20Singh&amp;author=A.%20Checinska%20Sielaff&amp;volume=5&amp;issue=81&amp;publication_year=2017&amp;pmid=28716113&amp;doi=10.1186/s40168-017-0292-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B5">
<cite>Bhatty M., Cruz M. R., Frank K. L., Laverde Gomez J. A., Andrade F., Garsin D. A., et al.  (2015). <em>Enterococcus faecalis</em> pCF 10-encoded surface proteins PrgA, PrgB (aggregation substance) and PrgC contribute to plasmid transfer, biofilm formation and virulence.
<em>Mol. Microbiol.</em>
95
660–677. 10.1111/mmi.12893
</cite> [<a href="https://doi.org/10.1111/mmi.12893" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4329047/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25431047/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Microbiol.&amp;title=Enterococcus%20faecalis%20pCF%2010-encoded%20surface%20proteins%20PrgA,%20PrgB%20(aggregation%20substance)%20and%20PrgC%20contribute%20to%20plasmid%20transfer,%20biofilm%20formation%20and%20virulence.&amp;author=M.%20Bhatty&amp;author=M.%20R.%20Cruz&amp;author=K.%20L.%20Frank&amp;author=J.%20A.%20Laverde%20Gomez&amp;author=F.%20Andrade&amp;volume=95&amp;publication_year=2015&amp;pages=660-677&amp;pmid=25431047&amp;doi=10.1111/mmi.12893&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B6">
<cite>Blaustein R. A., Mcfarland A. G., Maamar S. B., Lopez A., Castro-Wallace S., Hartmann E. M. (2019). Pangenomic approach to understanding microbial adaptations within a model built environment, the International Space Station, relative to human hosts and soil.
<em>mSystems</em>
4:e00281-18.</cite> [<a href="https://doi.org/10.1128/mSystems.00281-18" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6325168/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30637341/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=mSystems&amp;title=Pangenomic%20approach%20to%20understanding%20microbial%20adaptations%20within%20a%20model%20built%20environment,%20the%20International%20Space%20Station,%20relative%20to%20human%20hosts%20and%20soil.&amp;author=R.%20A.%20Blaustein&amp;author=A.%20G.%20Mcfarland&amp;author=S.%20B.%20Maamar&amp;author=A.%20Lopez&amp;author=S.%20Castro-Wallace&amp;volume=4&amp;issue=e00281-18&amp;publication_year=2019&amp;pmid=30637341&amp;doi=10.1128/mSystems.00281-18&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B7">
<cite>Bourgogne A., Garsin D. A., Qin X., Singh K. V., Sillanpaa J., Yerrapragada S., et al.  (2008). Large scale variation in <em>Enterococcus faecalis</em> illustrated by the genome analysis of strain OG1RF.
<em>Genome Biol.</em>
9:R110.</cite> [<a href="https://doi.org/10.1186/gb-2008-9-7-r110" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2530867/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18611278/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biol.&amp;title=Large%20scale%20variation%20in%20Enterococcus%20faecalis%20illustrated%20by%20the%20genome%20analysis%20of%20strain%20OG1RF.&amp;author=A.%20Bourgogne&amp;author=D.%20A.%20Garsin&amp;author=X.%20Qin&amp;author=K.%20V.%20Singh&amp;author=J.%20Sillanpaa&amp;volume=9&amp;issue=R110&amp;publication_year=2008&amp;pmid=18611278&amp;doi=10.1186/gb-2008-9-7-r110&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B100">
<cite>Bryan N. C., Christner B. C., Guzik T. G., Granger D. J., Stewart M. F. (2019). Abundance and survival of microbial aerosols in the troposphere and stratopshere.
<em>ISME J.</em>
13, 2789–2799. 10.1038/s41396-019-0474-0
</cite> [<a href="https://doi.org/10.1038/s41396-019-0474-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6794284/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31316133/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=ISME%20J.&amp;title=Abundance%20and%20survival%20of%20microbial%20aerosols%20in%20the%20troposphere%20and%20stratopshere.&amp;author=N.%20C.%20Bryan&amp;author=B.%20C.%20Christner&amp;author=T.%20G.%20Guzik&amp;author=D.%20J.%20Granger&amp;author=M.%20F.%20Stewart&amp;volume=13&amp;publication_year=2019&amp;pages=2789-2799&amp;pmid=31316133&amp;doi=10.1038/s41396-019-0474-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B8">
<cite>Burton A. S., Stahl S. E., John K. K., Jain M., Juul S., Turner D. J., et al.  (2020). Off earth identification of bacterial populations using 16S rDNA Nanopore Sequencing.
<em>Genes</em>
11:76. 10.3390/genes11010076
</cite> [<a href="https://doi.org/10.3390/genes11010076" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7016637/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31936690/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genes&amp;title=Off%20earth%20identification%20of%20bacterial%20populations%20using%2016S%20rDNA%20Nanopore%20Sequencing.&amp;author=A.%20S.%20Burton&amp;author=S.%20E.%20Stahl&amp;author=K.%20K.%20John&amp;author=M.%20Jain&amp;author=S.%20Juul&amp;volume=11&amp;issue=76&amp;publication_year=2020&amp;pmid=31936690&amp;doi=10.3390/genes11010076&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B9">
<cite>Carattoli A., Zankari E., Garcìa-Fernandez A., Larsen M. V., Lund O., Villa L., et al.  (2014). PlasmidFinder and pMLST: <em>in silico</em> detection and typing of plasmids.
<em>Antimicrob. Agents Chemother.</em>
58
2412–2414.</cite> [<a href="https://doi.org/10.1128/AAC.02412-14" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4068535/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24777092/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Antimicrob.%20Agents%20Chemother.&amp;title=PlasmidFinder%20and%20pMLST:%20in%20silico%20detection%20and%20typing%20of%20plasmids.&amp;author=A.%20Carattoli&amp;author=E.%20Zankari&amp;author=A.%20Garc%C3%ACa-Fernandez&amp;author=M.%20V.%20Larsen&amp;author=O.%20Lund&amp;volume=58&amp;publication_year=2014&amp;pages=2412-2414&amp;pmid=24777092&amp;doi=10.1128/AAC.02412-14&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B10">
<cite>Castro V., Thrasher A. N., Healy M., Ott C. M., Pierson D. L. (2004). Microbial characterization during the early habitation of the International Space Station.
<em>Microb. Ecol.</em>
47
119–126. 10.1007/s00248-003-1030-y
</cite> [<a href="https://doi.org/10.1007/s00248-003-1030-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14749908/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microb.%20Ecol.&amp;title=Microbial%20characterization%20during%20the%20early%20habitation%20of%20the%20International%20Space%20Station.&amp;author=V.%20Castro&amp;author=A.%20N.%20Thrasher&amp;author=M.%20Healy&amp;author=C.%20M.%20Ott&amp;author=D.%20L.%20Pierson&amp;volume=47&amp;publication_year=2004&amp;pages=119-126&amp;pmid=14749908&amp;doi=10.1007/s00248-003-1030-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B11">
<cite>Castro-Wallace S. L., Chiu C. Y., John K. K., Stahl S. E., Rubins K. H., Mcintyre A. B., et al.  (2017). Nanopore DNA sequencing and genome assembly on the International Space Station.
<em>Sci. Rep.</em>
7:18022.</cite> [<a href="https://doi.org/10.1038/s41598-017-18364-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5740133/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29269933/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&amp;title=Nanopore%20DNA%20sequencing%20and%20genome%20assembly%20on%20the%20International%20Space%20Station.&amp;author=S.%20L.%20Castro-Wallace&amp;author=C.%20Y.%20Chiu&amp;author=K.%20K.%20John&amp;author=S.%20E.%20Stahl&amp;author=K.%20H.%20Rubins&amp;volume=7&amp;issue=18022&amp;publication_year=2017&amp;pmid=29269933&amp;doi=10.1038/s41598-017-18364-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B12">
<cite>Chiu C. Y., Miller S. A. (2019). Clinical metagenomics.
<em>Nat. Rev. Genet.</em>
20
341–355.</cite> [<a href="https://doi.org/10.1038/s41576-019-0113-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6858796/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30918369/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Genet.&amp;title=Clinical%20metagenomics.&amp;author=C.%20Y.%20Chiu&amp;author=S.%20A.%20Miller&amp;volume=20&amp;publication_year=2019&amp;pages=341-355&amp;pmid=30918369&amp;doi=10.1038/s41576-019-0113-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B13">
<cite>CLSI (2017). <em>Performance Standards for Antimicrobial Susceptibility Testing.</em>
Wayne, PA: Clinical and Laboratory Standards Institute.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Performance%20Standards%20for%20Antimicrobial%20Susceptibility%20Testing.&amp;publication_year=2017&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B14">
<cite>Coburn P. S., Gilmore M. S. (2003). The <em>Enterococcus faecalis</em> cytolysin: a novel toxin active against eukaryotic and prokaryotic cells.
<em>Cell. Microbiol.</em>
5
661–669. 10.1046/j.1462-5822.2003.00310.x
</cite> [<a href="https://doi.org/10.1046/j.1462-5822.2003.00310.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12969372/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell.%20Microbiol.&amp;title=The%20Enterococcus%20faecalis%20cytolysin:%20a%20novel%20toxin%20active%20against%20eukaryotic%20and%20prokaryotic%20cells.&amp;author=P.%20S.%20Coburn&amp;author=M.%20S.%20Gilmore&amp;volume=5&amp;publication_year=2003&amp;pages=661-669&amp;pmid=12969372&amp;doi=10.1046/j.1462-5822.2003.00310.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B15">
<cite>Cosentino S., Larsen M. V., Aarestrup F. M., Lund O. (2013). PathogenFinder-distinguishing friend from foe using bacterial whole genome sequence data.
<em>PLoS One</em>
8:e77302. 10.1371/journal.pone.0077302
</cite> [<a href="https://doi.org/10.1371/journal.pone.0077302" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3810466/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24204795/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=PathogenFinder-distinguishing%20friend%20from%20foe%20using%20bacterial%20whole%20genome%20sequence%20data.&amp;author=S.%20Cosentino&amp;author=M.%20V.%20Larsen&amp;author=F.%20M.%20Aarestrup&amp;author=O.%20Lund&amp;volume=8&amp;issue=e77302&amp;publication_year=2013&amp;pmid=24204795&amp;doi=10.1371/journal.pone.0077302&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B16">
<cite>Couvin D., Bernheim A., Toffano-Nioche C., Touchon M., Michalik J., Néron B., et al.  (2018). CRISPRCasFinder, an update of CRISRFinder, includes a portable version, enhanced performance and integrates search for Cas proteins.
<em>Nucleic Acids Res.</em>
46
W246–W251.</cite> [<a href="https://doi.org/10.1093/nar/gky425" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6030898/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29790974/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=CRISPRCasFinder,%20an%20update%20of%20CRISRFinder,%20includes%20a%20portable%20version,%20enhanced%20performance%20and%20integrates%20search%20for%20Cas%20proteins.&amp;author=D.%20Couvin&amp;author=A.%20Bernheim&amp;author=C.%20Toffano-Nioche&amp;author=M.%20Touchon&amp;author=J.%20Michalik&amp;volume=46&amp;publication_year=2018&amp;pages=W246-W251&amp;pmid=29790974&amp;doi=10.1093/nar/gky425&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B17">
<cite>Crucian B., Stowe R., Mehta S., Uchakin P., Quiriarte H., Pierson D., et al.  (2013). Immune system dysregulation occurs during short duration spaceflight on board the space shuttle.
<em>J. Clin. Immunol.</em>
33
456–465. 10.1007/s10875-012-9824-7
</cite> [<a href="https://doi.org/10.1007/s10875-012-9824-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23100144/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Immunol.&amp;title=Immune%20system%20dysregulation%20occurs%20during%20short%20duration%20spaceflight%20on%20board%20the%20space%20shuttle.&amp;author=B.%20Crucian&amp;author=R.%20Stowe&amp;author=S.%20Mehta&amp;author=P.%20Uchakin&amp;author=H.%20Quiriarte&amp;volume=33&amp;publication_year=2013&amp;pages=456-465&amp;pmid=23100144&amp;doi=10.1007/s10875-012-9824-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B18">
<cite>Crucian B., Stowe R. P., Mehta S., Quiriarte H., Pierson D., Sams C. (2015). Alterations in adaptive immunity persist during long-duration spaceflight.
<em>NPJ Microgravity</em>
1:15013.</cite> [<a href="https://doi.org/10.1038/npjmgrav.2015.13" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5515498/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28725716/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=NPJ%20Microgravity&amp;title=Alterations%20in%20adaptive%20immunity%20persist%20during%20long-duration%20spaceflight.&amp;author=B.%20Crucian&amp;author=R.%20P.%20Stowe&amp;author=S.%20Mehta&amp;author=H.%20Quiriarte&amp;author=D.%20Pierson&amp;volume=1&amp;issue=15013&amp;publication_year=2015&amp;pmid=28725716&amp;doi=10.1038/npjmgrav.2015.13&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B19">
<cite>De Coster W., D’hert S., Schultz D. T., Cruts M., Van Broeckhoven C. (2018). NanoPack: visualizing and processing long-read sequencing data.
<em>Bioinformatics</em>
34
2666–2669. 10.1093/bioinformatics/bty149
</cite> [<a href="https://doi.org/10.1093/bioinformatics/bty149" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6061794/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29547981/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=NanoPack:%20visualizing%20and%20processing%20long-read%20sequencing%20data.&amp;author=W.%20De%20Coster&amp;author=S.%20D%E2%80%99hert&amp;author=D.%20T.%20Schultz&amp;author=M.%20Cruts&amp;author=C.%20Van%20Broeckhoven&amp;volume=34&amp;publication_year=2018&amp;pages=2666-2669&amp;pmid=29547981&amp;doi=10.1093/bioinformatics/bty149&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B20">
<cite>Ehrenfeld E., Clewell D. (1987). Transfer functions of the <em>Streptococcus faecalis</em> plasmid pAD1: organization of plasmid DNA encoding response to sex pheromone.
<em>J. Bacteriol.</em>
169
3473–3481. 10.1128/jb.169.8.3473-3481.1987
</cite> [<a href="https://doi.org/10.1128/jb.169.8.3473-3481.1987" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC212420/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/3038841/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Bacteriol.&amp;title=Transfer%20functions%20of%20the%20Streptococcus%20faecalis%20plasmid%20pAD1:%20organization%20of%20plasmid%20DNA%20encoding%20response%20to%20sex%20pheromone.&amp;author=E.%20Ehrenfeld&amp;author=D.%20Clewell&amp;volume=169&amp;publication_year=1987&amp;pages=3473-3481&amp;pmid=3038841&amp;doi=10.1128/jb.169.8.3473-3481.1987&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B21">
<cite>Fiore E., Van Tyne D., Gilmore M. S. (2019). Pathogenicity of Enterococci.
<em>Microbiol. Spectr.</em>
7:10.1128/microbiolspec.GPP3-0053-2018.</cite> [<a href="https://doi.org/10.1128/microbiolspec.gpp3-0053-2018" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6629438/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31298205/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microbiol.%20Spectr.&amp;title=Pathogenicity%20of%20Enterococci.&amp;author=E.%20Fiore&amp;author=D.%20Van%20Tyne&amp;author=M.%20S.%20Gilmore&amp;volume=7&amp;issue=10.1128/microbiolspec.GPP3-0053-2018&amp;publication_year=2019&amp;pmid=31298205&amp;doi=10.1128/microbiolspec.gpp3-0053-2018&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B22">
<cite>Gargis A. S., Cherney B., Conley A. B., Mclaughlin H. P., Sue D. (2019). Rapid detection of genetic engineering, structural variation, and antimicrobial resistance markers in bacterial biothreat pathogens by Nanopore sequencing.
<em>Sci. Rep.</em>
9:13501.</cite> [<a href="https://doi.org/10.1038/s41598-019-49700-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6751186/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31534162/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&amp;title=Rapid%20detection%20of%20genetic%20engineering,%20structural%20variation,%20and%20antimicrobial%20resistance%20markers%20in%20bacterial%20biothreat%20pathogens%20by%20Nanopore%20sequencing.&amp;author=A.%20S.%20Gargis&amp;author=B.%20Cherney&amp;author=A.%20B.%20Conley&amp;author=H.%20P.%20Mclaughlin&amp;author=D.%20Sue&amp;volume=9&amp;issue=13501&amp;publication_year=2019&amp;pmid=31534162&amp;doi=10.1038/s41598-019-49700-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B23">
<cite>Garsin D. A., Sifri C. D., Mylonakis E., Qin X., Singh K. V., Murray B. E., et al.  (2001). A simple model host for identifying Gram-positive virulence factors.
<em>Proc. Natl. Acad. Sci. U.S.A.</em>
98
10892–10897. 10.1073/pnas.191378698
</cite> [<a href="https://doi.org/10.1073/pnas.191378698" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC58570/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11535834/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A.&amp;title=A%20simple%20model%20host%20for%20identifying%20Gram-positive%20virulence%20factors.&amp;author=D.%20A.%20Garsin&amp;author=C.%20D.%20Sifri&amp;author=E.%20Mylonakis&amp;author=X.%20Qin&amp;author=K.%20V.%20Singh&amp;volume=98&amp;publication_year=2001&amp;pages=10892-10897&amp;pmid=11535834&amp;doi=10.1073/pnas.191378698&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B24">
<cite>Hammond T. G., Stodieck L., Birdsall H. H., Becker J. L., Koenig P., Hammond J. S., et al.  (2013). Effects of microgravity on the virulence of <em>Listeria monocytogenes</em>, <em>Enterococcus faecalis</em>, <em>Candida albicans</em>, and methicillin-resistant <em>Staphylococcus aureus</em>.
<em>Astrobiology</em>
13, 1081–1090.</cite> [<a href="https://doi.org/10.1089/ast.2013.0986" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24283929/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Astrobiology&amp;title=Effects%20of%20microgravity%20on%20the%20virulence%20of%20Listeria%20monocytogenes,%20Enterococcus%20faecalis,%20Candida%20albicans,%20and%20methicillin-resistant%20Staphylococcus%20aureus.&amp;author=T.%20G.%20Hammond&amp;author=L.%20Stodieck&amp;author=H.%20H.%20Birdsall&amp;author=J.%20L.%20Becker&amp;author=P.%20Koenig&amp;volume=13&amp;publication_year=2013&amp;pages=1081-1090&amp;pmid=24283929&amp;doi=10.1089/ast.2013.0986&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B25">
<cite>Han S. K., Lee D., Lee H., Kim D., Son H. G., Yang J.-S., et al.  (2016). OASIS 2: online application for survival analysis 2 with features for the analysis of maximal lifespan and healthspan in aging research.
<em>Oncotarget</em>
7
56147–56152. 10.18632/oncotarget.11269
</cite> [<a href="https://doi.org/10.18632/oncotarget.11269" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5302902/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27528229/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Oncotarget&amp;title=OASIS%202:%20online%20application%20for%20survival%20analysis%202%20with%20features%20for%20the%20analysis%20of%20maximal%20lifespan%20and%20healthspan%20in%20aging%20research.&amp;author=S.%20K.%20Han&amp;author=D.%20Lee&amp;author=H.%20Lee&amp;author=D.%20Kim&amp;author=H.%20G.%20Son&amp;volume=7&amp;publication_year=2016&amp;pages=56147-56152&amp;pmid=27528229&amp;doi=10.18632/oncotarget.11269&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B26">
<cite>Hussain M., Hastings J. G. M., White P. J. (1991). A chemically defined medium for slime production by coagulase-negative staphylococci.
<em>J. Med. Microbiol.</em>
34
143–147. 10.1099/00222615-34-3-143
</cite> [<a href="https://doi.org/10.1099/00222615-34-3-143" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2010904/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Med.%20Microbiol.&amp;title=A%20chemically%20defined%20medium%20for%20slime%20production%20by%20coagulase-negative%20staphylococci.&amp;author=M.%20Hussain&amp;author=J.%20G.%20M.%20Hastings&amp;author=P.%20J.%20White&amp;volume=34&amp;publication_year=1991&amp;pages=143-147&amp;pmid=2010904&amp;doi=10.1099/00222615-34-3-143&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B27">
<cite>Huycke M. M., Spiegel C. A., Gilmore M. S. (1991). Bacteremia caused by hemolytic, high-level gentamicin-resistant <em>Enterococcus faecalis</em>.
<em>Antimicrob. Agents Chemother.</em>
35
1626–1634. 10.1128/aac.35.8.1626
</cite> [<a href="https://doi.org/10.1128/aac.35.8.1626" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC245231/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1929336/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Antimicrob.%20Agents%20Chemother.&amp;title=Bacteremia%20caused%20by%20hemolytic,%20high-level%20gentamicin-resistant%20Enterococcus%20faecalis.&amp;author=M.%20M.%20Huycke&amp;author=C.%20A.%20Spiegel&amp;author=M.%20S.%20Gilmore&amp;volume=35&amp;publication_year=1991&amp;pages=1626-1634&amp;pmid=1929336&amp;doi=10.1128/aac.35.8.1626&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B28">
<cite>Ichijo T., Yamaguchi N., Tanigaki F., Shirakawa M., Nasu M. (2016). Four-year bacterial monitoring in the International Space Station—Japanese Experiment Module “Kibo” with culture-independent approach.
<em>NPJ Microgravity</em>
2:16007.</cite> [<a href="https://doi.org/10.1038/npjmgrav.2016.7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5515537/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28725725/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=NPJ%20Microgravity&amp;title=Four-year%20bacterial%20monitoring%20in%20the%20International%20Space%20Station%E2%80%94Japanese%20Experiment%20Module%20%E2%80%9CKibo%E2%80%9D%20with%20culture-independent%20approach.&amp;author=T.%20Ichijo&amp;author=N.%20Yamaguchi&amp;author=F.%20Tanigaki&amp;author=M.%20Shirakawa&amp;author=M.%20Nasu&amp;volume=2&amp;issue=16007&amp;publication_year=2016&amp;pmid=28725725&amp;doi=10.1038/npjmgrav.2016.7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B29">
<cite>ISS-MORD (2013). <em>International Space Station Medical Operatinos Requirements Documents</em>, Revision B Edn, ed.
Program I. S. S. (Houston, TX: National Aeronautics and Space Administration; ).</cite> [<a href="https://scholar.google.com/scholar_lookup?title=International%20Space%20Station%20Medical%20Operatinos%20Requirements%20Documents&amp;publication_year=2013&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B30">
<cite>Jett B. D., Hatter K. L., Huycke M. M., Gilmore M. S. (1997). Simplified agar plate method for quantifying viable bacteria.
<em>Biotechniques</em>
23
648–650. 10.2144/97234bm22
</cite> [<a href="https://doi.org/10.2144/97234bm22" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9343684/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biotechniques&amp;title=Simplified%20agar%20plate%20method%20for%20quantifying%20viable%20bacteria.&amp;author=B.%20D.%20Jett&amp;author=K.%20L.%20Hatter&amp;author=M.%20M.%20Huycke&amp;author=M.%20S.%20Gilmore&amp;volume=23&amp;publication_year=1997&amp;pages=648-650&amp;pmid=9343684&amp;doi=10.2144/97234bm22&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B31">
<cite>Jia B., Raphenya A. R., Alcock B., Waglechner N., Guo P., Tsang K. K., et al.  (2016). CARD 2017: expansion and model-centric curation of the comprehensive antibiotic resistance database.
<em>Nucleic Acids Res.</em>
45
D566–D573.</cite> [<a href="https://doi.org/10.1093/nar/gkw1004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5210516/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27789705/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=CARD%202017:%20expansion%20and%20model-centric%20curation%20of%20the%20comprehensive%20antibiotic%20resistance%20database.&amp;author=B.%20Jia&amp;author=A.%20R.%20Raphenya&amp;author=B.%20Alcock&amp;author=N.%20Waglechner&amp;author=P.%20Guo&amp;volume=45&amp;publication_year=2016&amp;pages=D566-D573&amp;pmid=27789705&amp;doi=10.1093/nar/gkw1004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B32">
<cite>JMP<sup>®</sup> (2019). <em>JMP® Pro</em>, 14.3 Edn
Cary, NC: SAS Institute Inc.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=JMP%C2%AE%20Pro&amp;publication_year=2019&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B33">
<cite>Joensen K. G., Scheutz F., Lund O., Hasman H., Kaas R. S., Nielsen E. M., et al.  (2014). Real-time whole-genome sequencing for routine typing, surveillance, and outbreak detection of verotoxigenic <em>Escherichia coli</em>.
<em>J. Clin. Microbiol.</em>
52
1501–1510. 10.1128/jcm.03617-13
</cite> [<a href="https://doi.org/10.1128/jcm.03617-13" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3993690/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24574290/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Microbiol.&amp;title=Real-time%20whole-genome%20sequencing%20for%20routine%20typing,%20surveillance,%20and%20outbreak%20detection%20of%20verotoxigenic%20Escherichia%20coli.&amp;author=K.%20G.%20Joensen&amp;author=F.%20Scheutz&amp;author=O.%20Lund&amp;author=H.%20Hasman&amp;author=R.%20S.%20Kaas&amp;volume=52&amp;publication_year=2014&amp;pages=1501-1510&amp;pmid=24574290&amp;doi=10.1128/jcm.03617-13&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B34">
<cite>Kim W., Tengra F. K., Young Z., Shong J., Marchand N., Chan H. K., et al.  (2013). Spaceflight promotes biofilm formation by <em>Pseudomonas aeruginosa</em>.
<em>PloS One</em>
8:e62437.</cite> [<a href="https://doi.org/10.1371/journal.pone.0062437" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3639165/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23658630/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PloS%20One&amp;title=Spaceflight%20promotes%20biofilm%20formation%20by%20Pseudomonas%20aeruginosa.&amp;author=W.%20Kim&amp;author=F.%20K.%20Tengra&amp;author=Z.%20Young&amp;author=J.%20Shong&amp;author=N.%20Marchand&amp;volume=8&amp;issue=e62437&amp;publication_year=2013&amp;pmid=23658630&amp;doi=10.1371/journal.pone.0062437&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B35">
<cite>Lang J. M., Coil D. A., Neches R. Y., Brown W. E., Cavalier D., Severance M., et al.  (2017). A microbial survey of the International Space Station (ISS).
<em>PeerJ</em>
5:e4029.</cite> [<a href="https://doi.org/10.7717/peerj.4029" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5827671/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29492330/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PeerJ&amp;title=A%20microbial%20survey%20of%20the%20International%20Space%20Station%20(ISS).&amp;author=J.%20M.%20Lang&amp;author=D.%20A.%20Coil&amp;author=R.%20Y.%20Neches&amp;author=W.%20E.%20Brown&amp;author=D.%20Cavalier&amp;volume=5&amp;issue=e4029&amp;publication_year=2017&amp;pmid=29492330&amp;doi=10.7717/peerj.4029&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B36">
<cite>Larsen M. V., Cosentino S., Rasmussen S., Friis C., Hasman H., Marvig R. L., et al.  (2012). Multilocus sequence typing of total-genome-sequenced bacteria.
<em>J. Clin. Microbiol.</em>
50
1355–1361. 10.1128/jcm.06094-11
</cite> [<a href="https://doi.org/10.1128/jcm.06094-11" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3318499/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22238442/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Microbiol.&amp;title=Multilocus%20sequence%20typing%20of%20total-genome-sequenced%20bacteria.&amp;author=M.%20V.%20Larsen&amp;author=S.%20Cosentino&amp;author=S.%20Rasmussen&amp;author=C.%20Friis&amp;author=H.%20Hasman&amp;volume=50&amp;publication_year=2012&amp;pages=1355-1361&amp;pmid=22238442&amp;doi=10.1128/jcm.06094-11&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B37">
<cite>Lebreton F., Manson A. L., Saavedra J. T., Straub T. J., Earl A. M., Gilmore M. S. (2017). Tracing the Enterococci from Paleozoic origins to the hospital.
<em>Cell</em>
169
849. 10.1016/j.cell.2017.04.027
</cite> [<a href="https://doi.org/10.1016/j.cell.2017.04.027" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5499534/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28502769/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Tracing%20the%20Enterococci%20from%20Paleozoic%20origins%20to%20the%20hospital.&amp;author=F.%20Lebreton&amp;author=A.%20L.%20Manson&amp;author=J.%20T.%20Saavedra&amp;author=T.%20J.%20Straub&amp;author=A.%20M.%20Earl&amp;volume=169&amp;issue=849&amp;publication_year=2017&amp;pmid=28502769&amp;doi=10.1016/j.cell.2017.04.027&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B38">
<cite>Maurer F. P., Christner M., Hentschke M., Rohde H. (2017). Advances in rapid identification and susceptibility testing of bacteria in the clinical microbiology laboratory: implications for patient care and antimicrobial stewardship programs.
<em>Infect. Dis. Rep.</em>
9
6839–6839.</cite> [<a href="https://doi.org/10.4081/idr.2017.6839" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5391540/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28458798/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Infect.%20Dis.%20Rep.&amp;title=Advances%20in%20rapid%20identification%20and%20susceptibility%20testing%20of%20bacteria%20in%20the%20clinical%20microbiology%20laboratory:%20implications%20for%20patient%20care%20and%20antimicrobial%20stewardship%20programs.&amp;author=F.%20P.%20Maurer&amp;author=M.%20Christner&amp;author=M.%20Hentschke&amp;author=H.%20Rohde&amp;volume=9&amp;publication_year=2017&amp;pages=6839-6839&amp;pmid=28458798&amp;doi=10.4081/idr.2017.6839&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B39">
<cite>McBride S. M., Fischetti V. A., Leblanc D. J., Moellering R. C., Jr., Gilmore M. S. (2007). Genetic diversity among <em>Enterococcus faecalis</em>.
<em>PLoS One</em>
2:e582. 10.1371/journal.pone.0000582
</cite> [<a href="https://doi.org/10.1371/journal.pone.0000582" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1899230/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17611618/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Genetic%20diversity%20among%20Enterococcus%20faecalis.&amp;author=S.%20M.%20McBride&amp;author=V.%20A.%20Fischetti&amp;author=D.%20J.%20Leblanc&amp;author=R.%20C.%20Moellering&amp;author=M.%20S.%20Gilmore&amp;volume=2&amp;issue=e582&amp;publication_year=2007&amp;pmid=17611618&amp;doi=10.1371/journal.pone.0000582&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B40">
<cite>Mclean R. J., Cassanto J. M., Barnes M. B., Koo J. H. (2001). Bacterial biofilm formation under microgravity. <em>FEMS Microbiol. Lett</em>. 195, 115–119.</cite> [<a href="https://doi.org/10.1111/j.1574-6968.2001.tb10507.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11179638/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=FEMS%20Microbiol.%20Lett&amp;title=Bacterial%20biofilm%20formation%20under%20microgravity&amp;author=R.%20J.%20Mclean&amp;author=J.%20M.%20Cassanto&amp;author=M.%20B.%20Barnes&amp;author=J.%20H.%20Koo&amp;volume=195&amp;publication_year=2001&amp;pages=115-119&amp;pmid=11179638&amp;doi=10.1111/j.1574-6968.2001.tb10507.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B41">
<cite>Mermel L. A. (2012). Infection prevention and control during prolonged human space travel.
<em>Clin. Infect. Dis.</em>
56
123–130. 10.1093/cid/cis861
</cite> [<a href="https://doi.org/10.1093/cid/cis861" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23051761/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Clin.%20Infect.%20Dis.&amp;title=Infection%20prevention%20and%20control%20during%20prolonged%20human%20space%20travel.&amp;author=L.%20A.%20Mermel&amp;volume=56&amp;publication_year=2012&amp;pages=123-130&amp;pmid=23051761&amp;doi=10.1093/cid/cis861&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B42">
<cite>Miller S., Naccache S. N., Samayoa E., Messacar K., Arevalo S., Federman S., et al.  (2019). Laboratory validation of a clinical metagenomic sequencing assay for pathogen detection in cerebrospinal fluid.
<em>Genome Res.</em>
29
831–842. 10.1101/gr.238170.118
</cite> [<a href="https://doi.org/10.1101/gr.238170.118" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6499319/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30992304/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Res.&amp;title=Laboratory%20validation%20of%20a%20clinical%20metagenomic%20sequencing%20assay%20for%20pathogen%20detection%20in%20cerebrospinal%20fluid.&amp;author=S.%20Miller&amp;author=S.%20N.%20Naccache&amp;author=E.%20Samayoa&amp;author=K.%20Messacar&amp;author=S.%20Arevalo&amp;volume=29&amp;publication_year=2019&amp;pages=831-842&amp;pmid=30992304&amp;doi=10.1101/gr.238170.118&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B43">
<cite>Mora M., Wink L., Kögler I., Mahnert A., Rettberg P., Schwendner P., et al.  (2019). Space Station conditions are selective but do not alter microbial characteristics relevant to human health.
<em>Nat. Commun.</em>
10:3990.</cite> [<a href="https://doi.org/10.1038/s41467-019-11682-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6728350/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31488812/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&amp;title=Space%20Station%20conditions%20are%20selective%20but%20do%20not%20alter%20microbial%20characteristics%20relevant%20to%20human%20health.&amp;author=M.%20Mora&amp;author=L.%20Wink&amp;author=I.%20K%C3%B6gler&amp;author=A.%20Mahnert&amp;author=P.%20Rettberg&amp;volume=10&amp;issue=3990&amp;publication_year=2019&amp;pmid=31488812&amp;doi=10.1038/s41467-019-11682-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B44">
<cite>Page A. J., Cummins C. A., Hunt M., Wong V. K., Reuter S., Holden M. T., et al.  (2015). Roary: rapid large-scale prokaryote pan genome analysis.
<em>Bioinformatics</em>
31
3691–3693. 10.1093/bioinformatics/btv421
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btv421" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4817141/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26198102/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Roary:%20rapid%20large-scale%20prokaryote%20pan%20genome%20analysis.&amp;author=A.%20J.%20Page&amp;author=C.%20A.%20Cummins&amp;author=M.%20Hunt&amp;author=V.%20K.%20Wong&amp;author=S.%20Reuter&amp;volume=31&amp;publication_year=2015&amp;pages=3691-3693&amp;pmid=26198102&amp;doi=10.1093/bioinformatics/btv421&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B45">
<cite>Parks D. H., Imelfort M., Skennerton C. T., Hugenholtz P., Tyson G. W. (2015). CheckM: assessing the quality of microbial genomes recovered from isolates, single cells, and metagenomes.
<em>Genome Res.</em>
25
1043–1055. 10.1101/gr.186072.114
</cite> [<a href="https://doi.org/10.1101/gr.186072.114" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4484387/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25977477/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Res.&amp;title=CheckM:%20assessing%20the%20quality%20of%20microbial%20genomes%20recovered%20from%20isolates,%20single%20cells,%20and%20metagenomes.&amp;author=D.%20H.%20Parks&amp;author=M.%20Imelfort&amp;author=C.%20T.%20Skennerton&amp;author=P.%20Hugenholtz&amp;author=G.%20W.%20Tyson&amp;volume=25&amp;publication_year=2015&amp;pages=1043-1055&amp;pmid=25977477&amp;doi=10.1101/gr.186072.114&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B46">
<cite>Paulsen I. T., Banerjei L., Myers G., Nelson K., Seshadri R., Read T. D., et al.  (2003). Role of mobile DNA in the evolution of vancomycin-resistant <em>Enterococcus faecalis</em>.
<em>Science</em>
299
2071–2074. 10.1126/science.1080613
</cite> [<a href="https://doi.org/10.1126/science.1080613" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12663927/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Role%20of%20mobile%20DNA%20in%20the%20evolution%20of%20vancomycin-resistant%20Enterococcus%20faecalis.&amp;author=I.%20T.%20Paulsen&amp;author=L.%20Banerjei&amp;author=G.%20Myers&amp;author=K.%20Nelson&amp;author=R.%20Seshadri&amp;volume=299&amp;publication_year=2003&amp;pages=2071-2074&amp;pmid=12663927&amp;doi=10.1126/science.1080613&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B47">
<cite>Powell J. R., Ausubel F. M. (2008). “Models of <em>Caenorhabditis elegans</em> infection by bacterial and fungal pathogens,” in <em>Innate Immunity</em>, ed.
Ewbank E. V. J. (Totowa, NJ: Humana Press Inc; ), 403–427. 10.1007/978-1-59745-570-1_24</cite> [<a href="https://doi.org/10.1007/978-1-59745-570-1_24" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18370168/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Innate%20Immunity&amp;author=J.%20R.%20Powell&amp;author=F.%20M.%20Ausubel&amp;publication_year=2008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B48">
<cite>Rooney B. V., Crucian B. E., Pierson D. L., Laudenslager M. L., Mehta S. K. (2019). Herpes virus reactivation in astronauts during spaceflight and its application on earth.
<em>Front. Microbiol.</em>
10:16. 10.3389/fmicb.2019.00016
</cite> [<a href="https://doi.org/10.3389/fmicb.2019.00016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6374706/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30792698/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front.%20Microbiol.&amp;title=Herpes%20virus%20reactivation%20in%20astronauts%20during%20spaceflight%20and%20its%20application%20on%20earth.&amp;author=B.%20V.%20Rooney&amp;author=B.%20E.%20Crucian&amp;author=D.%20L.%20Pierson&amp;author=M.%20L.%20Laudenslager&amp;author=S.%20K.%20Mehta&amp;volume=10&amp;issue=16&amp;publication_year=2019&amp;pmid=30792698&amp;doi=10.3389/fmicb.2019.00016&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B49">
<cite>Sahm D. F., Kissinger J., Gilmore M. S., Murray P. R., Mulder R., Solliday J., et al.  (1989). In vitro susceptibility studies of vancomycin-resistant <em>Enterococcus faecalis</em>.
<em>Antimicrob. Agents Chemother.</em>
33
1588–1591. 10.1128/aac.33.9.1588
</cite> [<a href="https://doi.org/10.1128/aac.33.9.1588" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC172707/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2554802/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Antimicrob.%20Agents%20Chemother.&amp;title=In%20vitro%20susceptibility%20studies%20of%20vancomycin-resistant%20Enterococcus%20faecalis.&amp;author=D.%20F.%20Sahm&amp;author=J.%20Kissinger&amp;author=M.%20S.%20Gilmore&amp;author=P.%20R.%20Murray&amp;author=R.%20Mulder&amp;volume=33&amp;publication_year=1989&amp;pages=1588-1591&amp;pmid=2554802&amp;doi=10.1128/aac.33.9.1588&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B50">
<cite>Sanderson N. D., Street T. L., Foster D., Swann J., Atkins B. L., Brent A. J., et al.  (2018). Real-time analysis of nanopore-based metagenomic sequencing from infected orthopaedic devices.
<em>BMC Genomics</em>
19:714. 10.1186/s12864-018-5094-y
</cite> [<a href="https://doi.org/10.1186/s12864-018-5094-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6161345/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30261842/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&amp;title=Real-time%20analysis%20of%20nanopore-based%20metagenomic%20sequencing%20from%20infected%20orthopaedic%20devices.&amp;author=N.%20D.%20Sanderson&amp;author=T.%20L.%20Street&amp;author=D.%20Foster&amp;author=J.%20Swann&amp;author=B.%20L.%20Atkins&amp;volume=19&amp;issue=714&amp;publication_year=2018&amp;pmid=30261842&amp;doi=10.1186/s12864-018-5094-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B51">
<cite>Schiwon K., Arends K., Rogowski K. M., Furch S., Prescha K., Sakinc T., et al.  (2013). Comparison of antibiotic resistance, biofilm formation and conjugative transfer of <em>Staphylococcus</em> and <em>Enterococcus</em> isolates from International Space Station and Antarctic Research Station Concordia.
<em>Microb. Ecol.</em>
65
638–651. 10.1007/s00248-013-0193-4
</cite> [<a href="https://doi.org/10.1007/s00248-013-0193-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23411852/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microb.%20Ecol.&amp;title=Comparison%20of%20antibiotic%20resistance,%20biofilm%20formation%20and%20conjugative%20transfer%20of%20Staphylococcus%20and%20Enterococcus%20isolates%20from%20International%20Space%20Station%20and%20Antarctic%20Research%20Station%20Concordia.&amp;author=K.%20Schiwon&amp;author=K.%20Arends&amp;author=K.%20M.%20Rogowski&amp;author=S.%20Furch&amp;author=K.%20Prescha&amp;volume=65&amp;publication_year=2013&amp;pages=638-651&amp;pmid=23411852&amp;doi=10.1007/s00248-013-0193-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B52">
<cite>Schloissnig S., Arumugam M., Sunagawa S., Mitreva M., Tap J., Zhu A., et al.  (2013). Genomic variation landscape of the human gut microbiome.
<em>Nature</em>
493
45–50. 10.1038/nature11711
</cite> [<a href="https://doi.org/10.1038/nature11711" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3536929/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23222524/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genomic%20variation%20landscape%20of%20the%20human%20gut%20microbiome.&amp;author=S.%20Schloissnig&amp;author=M.%20Arumugam&amp;author=S.%20Sunagawa&amp;author=M.%20Mitreva&amp;author=J.%20Tap&amp;volume=493&amp;publication_year=2013&amp;pages=45-50&amp;pmid=23222524&amp;doi=10.1038/nature11711&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B53">
<cite>Seemann T. (2014). Prokka: rapid prokaryotic genome annotation.
<em>Bioinformatics</em>
30
2068–2069. 10.1093/bioinformatics/btu153
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btu153" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24642063/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Prokka:%20rapid%20prokaryotic%20genome%20annotation.&amp;author=T.%20Seemann&amp;volume=30&amp;publication_year=2014&amp;pages=2068-2069&amp;pmid=24642063&amp;doi=10.1093/bioinformatics/btu153&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B54">
<cite>Shankar N., Baghdayan A. S., Gilmore M. S. (2002). Modulation of virulence within a pathogenicity island in vancomycin-resistant <em>Enterococcus faecalis</em>.
<em>Nature</em>
417
746–750. 10.1038/nature00802
</cite> [<a href="https://doi.org/10.1038/nature00802" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12066186/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Modulation%20of%20virulence%20within%20a%20pathogenicity%20island%20in%20vancomycin-resistant%20Enterococcus%20faecalis.&amp;author=N.%20Shankar&amp;author=A.%20S.%20Baghdayan&amp;author=M.%20S.%20Gilmore&amp;volume=417&amp;publication_year=2002&amp;pages=746-750&amp;pmid=12066186&amp;doi=10.1038/nature00802&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B55">
<cite>Sielaff A. C., Urbaniak C., Mohan G. B. M., Stepanov V. G., Tran Q., Wood J. M., et al.  (2019). Characterization of the total and viable bacterial and fungal communities associated with the International Space Station surfaces.
<em>Microbiome</em>
7:50.</cite> [<a href="https://doi.org/10.1186/s40168-019-0666-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6452512/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30955503/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microbiome&amp;title=Characterization%20of%20the%20total%20and%20viable%20bacterial%20and%20fungal%20communities%20associated%20with%20the%20International%20Space%20Station%20surfaces.&amp;author=A.%20C.%20Sielaff&amp;author=C.%20Urbaniak&amp;author=G.%20B.%20M.%20Mohan&amp;author=V.%20G.%20Stepanov&amp;author=Q.%20Tran&amp;volume=7&amp;issue=50&amp;publication_year=2019&amp;pmid=30955503&amp;doi=10.1186/s40168-019-0666-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B56">
<cite>Singh N. K., Bezdan D., Sielaff A. C., Wheeler K., Mason C. E., Venkateswaran K. (2018a). Multi-drug resistant <em>Enterobacter bugandensis</em> species isolated from the International Space Station and comparative genomic analyses with human pathogenic strains.
<em>BMC Microbiol.</em>
18:175. 10.1186/s12866-018-1325-2
</cite> [<a href="https://doi.org/10.1186/s12866-018-1325-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6251167/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30466389/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Microbiol.&amp;title=Multi-drug%20resistant%20Enterobacter%20bugandensis%20species%20isolated%20from%20the%20International%20Space%20Station%20and%20comparative%20genomic%20analyses%20with%20human%20pathogenic%20strains.&amp;author=N.%20K.%20Singh&amp;author=D.%20Bezdan&amp;author=A.%20C.%20Sielaff&amp;author=K.%20Wheeler&amp;author=C.%20E.%20Mason&amp;volume=18&amp;issue=175&amp;publication_year=2018a&amp;pmid=30466389&amp;doi=10.1186/s12866-018-1325-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B57">
<cite>Singh N. K., Wood J. M., Karouia F., Venkateswaran K. (2018b). Succession and persistence of microbial communities and antimicrobial resistance genes associated with International Space Station environmental surfaces.
<em>Microbiome</em>
6:204.</cite> [<a href="https://doi.org/10.1186/s40168-018-0585-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6234677/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30424821/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microbiome&amp;title=Succession%20and%20persistence%20of%20microbial%20communities%20and%20antimicrobial%20resistance%20genes%20associated%20with%20International%20Space%20Station%20environmental%20surfaces.&amp;author=N.%20K.%20Singh&amp;author=J.%20M.%20Wood&amp;author=F.%20Karouia&amp;author=K.%20Venkateswaran&amp;volume=6&amp;issue=204&amp;publication_year=2018b&amp;pmid=30424821&amp;doi=10.1186/s40168-018-0585-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B58">
<cite>Sobisch L.-Y., Rogowski K. M., Fuchs J., Schmieder W., Vaishampayan A., Oles P., et al.  (2019). Biofilm forming antibiotic resistant Gram-positive pathogens isolated from surfaces on the International Space Station.
<em>Front. Microbiol.</em>
10:543. 10.3389/fmicb.2019.00543
</cite> [<a href="https://doi.org/10.3389/fmicb.2019.00543" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6433718/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30941112/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front.%20Microbiol.&amp;title=Biofilm%20forming%20antibiotic%20resistant%20Gram-positive%20pathogens%20isolated%20from%20surfaces%20on%20the%20International%20Space%20Station.&amp;author=L.-Y.%20Sobisch&amp;author=K.%20M.%20Rogowski&amp;author=J.%20Fuchs&amp;author=W.%20Schmieder&amp;author=A.%20Vaishampayan&amp;volume=10&amp;issue=543&amp;publication_year=2019&amp;pmid=30941112&amp;doi=10.3389/fmicb.2019.00543&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B59">
<cite>Staley J. T., Konopka A. (1985). Measurement of in situ activities of nonphotosynthetic microorganisms in aquatic and terrestrial habitats.
<em>Annu. Rev. Microbiol.</em>
39
321–346. 10.1146/annurev.mi.39.100185.001541
</cite> [<a href="https://doi.org/10.1146/annurev.mi.39.100185.001541" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/3904603/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annu.%20Rev.%20Microbiol.&amp;title=Measurement%20of%20in%20situ%20activities%20of%20nonphotosynthetic%20microorganisms%20in%20aquatic%20and%20terrestrial%20habitats.&amp;author=J.%20T.%20Staley&amp;author=A.%20Konopka&amp;volume=39&amp;publication_year=1985&amp;pages=321-346&amp;pmid=3904603&amp;doi=10.1146/annurev.mi.39.100185.001541&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B60">
<cite>Su L., Zhou L., Liu J., Cen Z., Wu C., Wang T., et al.  (2014). Phenotypic, genomic, transcriptomic and proteomic changes in <em>Bacillus cereus</em> after a short-term space flight.
<em>Adv. Space Res.</em>
53
18–29. 10.1016/j.asr.2013.08.001</cite> [<a href="https://doi.org/10.1016/j.asr.2013.08.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Adv.%20Space%20Res.&amp;title=Phenotypic,%20genomic,%20transcriptomic%20and%20proteomic%20changes%20in%20Bacillus%20cereus%20after%20a%20short-term%20space%20flight.&amp;author=L.%20Su&amp;author=L.%20Zhou&amp;author=J.%20Liu&amp;author=Z.%20Cen&amp;author=C.%20Wu&amp;volume=53&amp;publication_year=2014&amp;pages=18-29&amp;doi=10.1016/j.asr.2013.08.001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B61">
<cite>Tirumalai M. R., Stepanov V. G., Wünsche A., Montazari S., Gonzalez R. O., Venkateswaran K., et al.  (2018). <em>Bacillus safensis</em> FO-36b and <em>Bacillus pumilus</em> SAFR-032: a whole genome comparison of two spacecraft assembly facility isolates.
<em>BMC Microbiol.</em>
18:57. 10.1186/s12866-018-1191-y
</cite> [<a href="https://doi.org/10.1186/s12866-018-1191-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5994023/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29884123/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Microbiol.&amp;title=Bacillus%20safensis%20FO-36b%20and%20Bacillus%20pumilus%20SAFR-032:%20a%20whole%20genome%20comparison%20of%20two%20spacecraft%20assembly%20facility%20isolates.&amp;author=M.%20R.%20Tirumalai&amp;author=V.%20G.%20Stepanov&amp;author=A.%20W%C3%BCnsche&amp;author=S.%20Montazari&amp;author=R.%20O.%20Gonzalez&amp;volume=18&amp;issue=57&amp;publication_year=2018&amp;pmid=29884123&amp;doi=10.1186/s12866-018-1191-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B62">
<cite>Urbaniak C., Sielaff A. C., Frey K., Allen J., Singh N., Jaing C., et al.  (2018). Detection of antimicrobial resistance genes associated with the International Space Station environmental surfaces.
<em>Sci. Rep.</em>
8:814.</cite> [<a href="https://doi.org/10.1038/s41598-017-18506-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5770469/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29339831/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&amp;title=Detection%20of%20antimicrobial%20resistance%20genes%20associated%20with%20the%20International%20Space%20Station%20environmental%20surfaces.&amp;author=C.%20Urbaniak&amp;author=A.%20C.%20Sielaff&amp;author=K.%20Frey&amp;author=J.%20Allen&amp;author=N.%20Singh&amp;volume=8&amp;issue=814&amp;publication_year=2018&amp;pmid=29339831&amp;doi=10.1038/s41598-017-18506-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B63">
<cite>Urbaniak C., Van Dam P., Zaborin A., Zaborina O., Gilbert J. A., Torok T., et al.  (2019). Genomic characterization and virulence potential of two <em>Fusarium oxysporum</em> isolates cultured from the International Space Station.
<em>mSystems</em>
4:e345-18.</cite> [<a href="https://doi.org/10.1128/mSystems.00345-18" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6426649/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30944876/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=mSystems&amp;title=Genomic%20characterization%20and%20virulence%20potential%20of%20two%20Fusarium%20oxysporum%20isolates%20cultured%20from%20the%20International%20Space%20Station.&amp;author=C.%20Urbaniak&amp;author=P.%20Van%20Dam&amp;author=A.%20Zaborin&amp;author=O.%20Zaborina&amp;author=J.%20A.%20Gilbert&amp;volume=4&amp;issue=e345-18&amp;publication_year=2019&amp;pmid=30944876&amp;doi=10.1128/mSystems.00345-18&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B64">
<cite>Van Tyne D., Manson A. L., Huycke M. M., Karanicolas J., Earl A. M., Gilmore M. S. (2019). Impact of antibiotic treatment and host innate immune pressure on enterococcal adaptation in the human bloodstream.
<em>Sci. Transl. Med.</em>
11:eaat8418. 10.1126/scitranslmed.aat8418
</cite> [<a href="https://doi.org/10.1126/scitranslmed.aat8418" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6681809/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30971455/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Transl.%20Med.&amp;title=Impact%20of%20antibiotic%20treatment%20and%20host%20innate%20immune%20pressure%20on%20enterococcal%20adaptation%20in%20the%20human%20bloodstream.&amp;author=D.%20Van%20Tyne&amp;author=A.%20L.%20Manson&amp;author=M.%20M.%20Huycke&amp;author=J.%20Karanicolas&amp;author=A.%20M.%20Earl&amp;volume=11&amp;issue=eaat8418&amp;publication_year=2019&amp;pmid=30971455&amp;doi=10.1126/scitranslmed.aat8418&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B65">
<cite>Vaughan T. G. (2017). IcyTree: rapid browser-based visualization for phlogenetic trees and networks.
<em>Bioinformatics</em>. 33, 2392–2394. 10.1093/bioinformatics/btx155
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btx155" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5860111/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28407035/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=IcyTree:%20rapid%20browser-based%20visualization%20for%20phlogenetic%20trees%20and%20networks.&amp;author=T.%20G.%20Vaughan&amp;volume=33&amp;publication_year=2017&amp;pages=2392-2394&amp;pmid=28407035&amp;doi=10.1093/bioinformatics/btx155&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B66">
<cite>Venkateswaran K. (2017). “Microbial Characteristics of ISS Environmental Surfaces,” in <em>Proceedings of the 47th International Conference on Environmental Systems</em>, Charleston, SC.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Proceedings%20of%20the%2047th%20International%20Conference%20on%20Environmental%20Systems&amp;author=K.%20Venkateswaran&amp;publication_year=2017&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B67">
<cite>Wang Y., Yuan Y., Liu J., Su L., Chang D., Guo Y., et al.  (2014). Transcriptomic and proteomic responses of <em>Serratia marcescens</em> to spaceflight conditions involve large-scale changes in metabolic pathways.
<em>Adv. Space Res.</em>
53
1108–1117. 10.1016/j.asr.2014.01.018</cite> [<a href="https://doi.org/10.1016/j.asr.2014.01.018" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Adv.%20Space%20Res.&amp;title=Transcriptomic%20and%20proteomic%20responses%20of%20Serratia%20marcescens%20to%20spaceflight%20conditions%20involve%20large-scale%20changes%20in%20metabolic%20pathways.&amp;author=Y.%20Wang&amp;author=Y.%20Yuan&amp;author=J.%20Liu&amp;author=L.%20Su&amp;author=D.%20Chang&amp;volume=53&amp;publication_year=2014&amp;pages=1108-1117&amp;doi=10.1016/j.asr.2014.01.018&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B68">
<cite>Weiner L. M., Webb A. K., Limbago B., Dudeck M. A., Patel J., Kallen A. J., et al.  (2016). Antimicrobial-resistant pathogens associated with healthcare-associated infections: summary of data reported to the National Healthcare Safety Network at the Centers for Disease Control and Prevention, 2011–2014.
<em>Infect. Control Hosp. Epidemiol.</em>
37
1288–1301. 10.1017/ice.2016.174
</cite> [<a href="https://doi.org/10.1017/ice.2016.174" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6857725/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27573805/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Infect.%20Control%20Hosp.%20Epidemiol.&amp;title=Antimicrobial-resistant%20pathogens%20associated%20with%20healthcare-associated%20infections:%20summary%20of%20data%20reported%20to%20the%20National%20Healthcare%20Safety%20Network%20at%20the%20Centers%20for%20Disease%20Control%20and%20Prevention,%202011%E2%80%932014.&amp;author=L.%20M.%20Weiner&amp;author=A.%20K.%20Webb&amp;author=B.%20Limbago&amp;author=M.%20A.%20Dudeck&amp;author=J.%20Patel&amp;volume=37&amp;publication_year=2016&amp;pages=1288-1301&amp;pmid=27573805&amp;doi=10.1017/ice.2016.174&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B69">
<cite>Wick R. R., Judd L. M., Gorrie C. L., Holt K. E. (2017). Unicycler: resolving bacterial genome assemblies from short and long sequencing reads.
<em>PLoS Comput. Biol.</em>
13:e1005595. 10.1371/journal.pcbi.1005595
</cite> [<a href="https://doi.org/10.1371/journal.pcbi.1005595" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5481147/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28594827/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Comput.%20Biol.&amp;title=Unicycler:%20resolving%20bacterial%20genome%20assemblies%20from%20short%20and%20long%20sequencing%20reads.&amp;author=R.%20R.%20Wick&amp;author=L.%20M.%20Judd&amp;author=C.%20L.%20Gorrie&amp;author=K.%20E.%20Holt&amp;volume=13&amp;issue=e1005595&amp;publication_year=2017&amp;pmid=28594827&amp;doi=10.1371/journal.pcbi.1005595&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B70">
<cite>Wilson J., Ott C., Zu Bentrup K. H., Ramamurthy R., Quick L., Porwollik S., et al.  (2007). Space flight alters bacterial gene expression and virulence and reveals a role for global regulator Hfq.
<em>Proc. Natl. Acad. Sci. U.S.A.</em>
104
16299–16304. 10.1073/pnas.0707155104
</cite> [<a href="https://doi.org/10.1073/pnas.0707155104" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2042201/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17901201/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A.&amp;title=Space%20flight%20alters%20bacterial%20gene%20expression%20and%20virulence%20and%20reveals%20a%20role%20for%20global%20regulator%20Hfq.&amp;author=J.%20Wilson&amp;author=C.%20Ott&amp;author=K.%20H.%20Zu%20Bentrup&amp;author=R.%20Ramamurthy&amp;author=L.%20Quick&amp;volume=104&amp;publication_year=2007&amp;pages=16299-16304&amp;pmid=17901201&amp;doi=10.1073/pnas.0707155104&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B71">
<cite>Yuen G. J., Ausubel F. M. (2018). Both live and dead Enterococci activate <em>Caenorhabditis elegans</em> host defense via immune and stress pathways.
<em>Virulence</em>
9
683–699. 10.1080/21505594.2018.1438025
</cite> [<a href="https://doi.org/10.1080/21505594.2018.1438025" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5955442/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29436902/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Virulence&amp;title=Both%20live%20and%20dead%20Enterococci%20activate%20Caenorhabditis%20elegans%20host%20defense%20via%20immune%20and%20stress%20pathways.&amp;author=G.%20J.%20Yuen&amp;author=F.%20M.%20Ausubel&amp;volume=9&amp;publication_year=2018&amp;pages=683-699&amp;pmid=29436902&amp;doi=10.1080/21505594.2018.1438025&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p">
<span>Supplementary Figure 1</span><p>Gene membership of the 51 <em>Enterococcus faecalis</em> isolate genomes analyzed. Counts and categories as estimated by Roary analysis of presence/absence as described in the methods.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/7829349/bin/Data_Sheet_2.PDF" data-ga-action="click_feat_suppl" class="usa-link">Click here for additional data file.</a><sup> (4MB, PDF) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p">
<span>Supplementary Figure 2</span><p>Sequence similarity of <em>Enterococcus faecalis</em> genes unique to ISS_1, ISS_2/ISS_3, and ISS_4. Query coverage and percentage sequence identity from BLASTn searches across all isolates (upper left) and for each isolate, respectively. Because ISS_2 and ISS_3 have no individually unique genes, and differ in their core gene alignment by only 14 bases, but have genes that are not represented in any of the other 49 genomes studied, ISS_3 was excluded from this analysis. <a href="#DS1" class="usa-link">Supplementary Data File 1</a> contains details of each BLASTn hit.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/7829349/bin/Data_Sheet_2.PDF" data-ga-action="click_feat_suppl" class="usa-link">Click here for additional data file.</a><sup> (4MB, PDF) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="caption p">
<span>Supplementary Figure 3</span><p>Sequence similarity of <em>Enterococcus faecalis</em> genes unique to ISS_1, ISS_2/ISS_3, and ISS_4 segmented by the genus of the hit subject. Query coverage and percentage sequence identity from BLASTn searches are as in <a href="#FS2" class="usa-link">Supplementary Figure S2</a> (upper left) after eliminating all hits associated with <em>Enterococcus</em> (upper left) and further segmenting by genus (there were only three remaining genera).</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/7829349/bin/Data_Sheet_2.PDF" data-ga-action="click_feat_suppl" class="usa-link">Click here for additional data file.</a><sup> (4MB, PDF) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="caption p">
<span>Supplementary Table 1</span><p>Illumina HiSeq short read coverage estimates.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/7829349/bin/Data_Sheet_2.PDF" data-ga-action="click_feat_suppl" class="usa-link">Click here for additional data file.</a><sup> (4MB, PDF) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="caption p">
<span>Supplementary Table 2</span><p>Nanopore long read coverage estimates.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/7829349/bin/Data_Sheet_2.PDF" data-ga-action="click_feat_suppl" class="usa-link">Click here for additional data file.</a><sup> (4MB, PDF) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="caption p">
<span>Supplementary Table 3</span><p>Predicted intact phage content as determined by PHASTER for ISS and control <em>E. faecalis</em> isolates (<a href="#B2" class="usa-link" aria-describedby="B2">Arndt et al., 2016</a>).</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/7829349/bin/Data_Sheet_2.PDF" data-ga-action="click_feat_suppl" class="usa-link">Click here for additional data file.</a><sup> (4MB, PDF) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material7_reqid_"><div class="caption p">
<span>Supplementary Table 4</span><p>CRISPR-Cas system detection in the genomes of ISS and reference strains (<a href="#B16" class="usa-link" aria-describedby="B16">Couvin et al., 2018</a>).</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/7829349/bin/Data_Sheet_2.PDF" data-ga-action="click_feat_suppl" class="usa-link">Click here for additional data file.</a><sup> (4MB, PDF) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material8_reqid_"><div class="caption p">
<span>Supplementary Data File 1</span><p>Excel file listing all genomes used in the project, the detected presence/absence of each annotated gene for all genomes, and BLASTn hits for genes unique to the ISS isolates.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/7829349/bin/Data_Sheet_1.XLSX" data-ga-action="click_feat_suppl" class="usa-link">Click here for additional data file.</a><sup> (2.4MB, XLSX) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The datasets generated for this study can be found in the NCBI BioProject website (<a href="https://www.ncbi.nlm.nih.gov/bioproject" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/bioproject</a>) using the BioProject ID: <a href="https://ncbi.nlm.nih.gov/nucleotide/PRJNA587161" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PRJNA587161</a> and the accession numbers <a href="https://ncbi.nlm.nih.gov/nucleotide/SAMN13182395" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">SAMN13182395</a>–<a href="https://ncbi.nlm.nih.gov/nucleotide/SAMN13182398" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">SAMN13182398</a>.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Frontiers in Microbiology are provided here courtesy of <strong>Frontiers Media SA</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.3389/fmicb.2020.515319"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/fmicb-11-515319.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (3.1 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/7829349/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/7829349/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC7829349%2F%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC7829349/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC7829349/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC7829349/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/33505359/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC7829349/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/33505359/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC7829349/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/7829349/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="GNDr2Jk4sAEnnGP7I9FABa1Ot4Yhpk81dF2qojwi82ltyDawZGZhTZAzQFmYAEyS">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
